{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "authorship_tag": "ABX9TyO9Smo8FMMxcapLy3DtD7Fy",
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/haririsina/colab_gene/blob/main/gene_data_collection.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "gIDGYJVqlkqJ",
        "outputId": "e61b10f1-b3d1-4f91-8cbe-3b20c7c6e5a6"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting GEOparse\n",
            "  Downloading GEOparse-2.0.3.tar.gz (278 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m278.5/278.5 KB\u001b[0m \u001b[31m7.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: numpy>=1.7 in /usr/local/lib/python3.8/dist-packages (from GEOparse) (1.21.6)\n",
            "Requirement already satisfied: pandas>=0.17 in /usr/local/lib/python3.8/dist-packages (from GEOparse) (1.3.5)\n",
            "Requirement already satisfied: requests>=2.21.0 in /usr/local/lib/python3.8/dist-packages (from GEOparse) (2.25.1)\n",
            "Requirement already satisfied: tqdm>=4.31.1 in /usr/local/lib/python3.8/dist-packages (from GEOparse) (4.64.1)\n",
            "Requirement already satisfied: python-dateutil>=2.7.3 in /usr/local/lib/python3.8/dist-packages (from pandas>=0.17->GEOparse) (2.8.2)\n",
            "Requirement already satisfied: pytz>=2017.3 in /usr/local/lib/python3.8/dist-packages (from pandas>=0.17->GEOparse) (2022.7)\n",
            "Requirement already satisfied: chardet<5,>=3.0.2 in /usr/local/lib/python3.8/dist-packages (from requests>=2.21.0->GEOparse) (4.0.0)\n",
            "Requirement already satisfied: urllib3<1.27,>=1.21.1 in /usr/local/lib/python3.8/dist-packages (from requests>=2.21.0->GEOparse) (1.24.3)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.8/dist-packages (from requests>=2.21.0->GEOparse) (2022.12.7)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.8/dist-packages (from requests>=2.21.0->GEOparse) (2.10)\n",
            "Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.7.3->pandas>=0.17->GEOparse) (1.15.0)\n",
            "Building wheels for collected packages: GEOparse\n",
            "  Building wheel for GEOparse (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for GEOparse: filename=GEOparse-2.0.3-py3-none-any.whl size=29064 sha256=2fe8edaebc314492b1ce79c9df43101837894dc54ef42eca0eb9af6d9532f299\n",
            "  Stored in directory: /root/.cache/pip/wheels/7a/75/14/727a857e321c5e5590d7e58efe9f028d753d340cea4ee540f0\n",
            "Successfully built GEOparse\n",
            "Installing collected packages: GEOparse\n",
            "Successfully installed GEOparse-2.0.3\n"
          ]
        }
      ],
      "source": [
        "! pip install GEOparse"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import GEOparse\n",
        "\n",
        "gse = GEOparse.get_GEO(geo=\"GSE99039\", destdir=\"./\")\n",
        "\n",
        "print()\n",
        "print(\"GSM example:\")\n",
        "gsm_list = []\n",
        "for gsm_name, gsm in gse.gsms.items():\n",
        "    print(\"Name: \", gsm_name)\n",
        "    print(\"Metadata:\",)\n",
        "    \n",
        "    meta_dict = {}\n",
        "    for key, value in gsm.metadata.items():\n",
        "        print(\" - %s : %s\" % (key, \", \".join(value)))\n",
        "        meta_dict[key] = value\n",
        "    \n",
        "    temp_dict = {\n",
        "        'name': gsm_name,\n",
        "        'metadata': meta_dict,\n",
        "        }\n",
        "    \n",
        "    gsm_list.append(temp_dict)\n",
        "\n",
        "    # print (\"Table data:\",)\n",
        "    # print (gsm.table.head())\n",
        "    # break\n",
        "\n",
        "print()\n",
        "print(\"GPL example:\")\n",
        "gpl_list = []\n",
        "for gpl_name, gpl in gse.gpls.items():\n",
        "    print(\"Name: \", gpl_name)\n",
        "    print(\"Metadata:\",)\n",
        "    \n",
        "    meta_dict = {}\n",
        "    for key, value in gpl.metadata.items():\n",
        "        print(\" - %s : %s\" % (key, \", \".join(value)))\n",
        "        meta_dict[key] = value\n",
        "\n",
        "        temp_dict = {\n",
        "        'name': gsm_name,\n",
        "        'metadata': meta_dict,\n",
        "        }\n",
        "    \n",
        "    gpl_list.append(temp_dict)\n",
        "\n",
        "    # print(\"Table data:\",)\n",
        "    # print(gpl.table.head())\n",
        "    # break\n",
        "\n",
        "print(len(gsm_list))\n",
        "print(gsm_list[0])\n",
        "print(len(gpl_list))\n",
        "print(gpl_list[0])\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "DQufj0tOlqJn",
        "outputId": "23f38b16-5a34-4986-c6fc-c892e475bc7f"
      },
      "execution_count": 5,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "06-Jan-2023 17:48:17 DEBUG utils - Directory ./ already exists. Skipping.\n",
            "DEBUG:GEOparse:Directory ./ already exists. Skipping.\n",
            "06-Jan-2023 17:48:17 INFO GEOparse - File already exist: using local version.\n",
            "INFO:GEOparse:File already exist: using local version.\n",
            "06-Jan-2023 17:48:17 INFO GEOparse - Parsing ./GSE99039_family.soft.gz: \n",
            "INFO:GEOparse:Parsing ./GSE99039_family.soft.gz: \n",
            "06-Jan-2023 17:48:17 DEBUG GEOparse - DATABASE: GeoMiame\n",
            "DEBUG:GEOparse:DATABASE: GeoMiame\n",
            "06-Jan-2023 17:48:17 DEBUG GEOparse - SERIES: GSE99039\n",
            "DEBUG:GEOparse:SERIES: GSE99039\n",
            "06-Jan-2023 17:48:17 DEBUG GEOparse - PLATFORM: GPL570\n",
            "DEBUG:GEOparse:PLATFORM: GPL570\n",
            "06-Jan-2023 17:48:21 DEBUG GEOparse - SAMPLE: GSM2630758\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630758\n",
            "06-Jan-2023 17:48:21 DEBUG GEOparse - SAMPLE: GSM2630759\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630759\n",
            "06-Jan-2023 17:48:21 DEBUG GEOparse - SAMPLE: GSM2630760\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630760\n",
            "06-Jan-2023 17:48:21 DEBUG GEOparse - SAMPLE: GSM2630761\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630761\n",
            "06-Jan-2023 17:48:21 DEBUG GEOparse - SAMPLE: GSM2630762\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630762\n",
            "06-Jan-2023 17:48:21 DEBUG GEOparse - SAMPLE: GSM2630763\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630763\n",
            "06-Jan-2023 17:48:21 DEBUG GEOparse - SAMPLE: GSM2630764\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630764\n",
            "06-Jan-2023 17:48:22 DEBUG GEOparse - SAMPLE: GSM2630765\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630765\n",
            "06-Jan-2023 17:48:22 DEBUG GEOparse - SAMPLE: GSM2630766\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630766\n",
            "06-Jan-2023 17:48:22 DEBUG GEOparse - SAMPLE: GSM2630767\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630767\n",
            "06-Jan-2023 17:48:22 DEBUG GEOparse - SAMPLE: GSM2630768\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630768\n",
            "06-Jan-2023 17:48:22 DEBUG GEOparse - SAMPLE: GSM2630769\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630769\n",
            "06-Jan-2023 17:48:22 DEBUG GEOparse - SAMPLE: GSM2630770\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630770\n",
            "06-Jan-2023 17:48:23 DEBUG GEOparse - SAMPLE: GSM2630771\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630771\n",
            "06-Jan-2023 17:48:23 DEBUG GEOparse - SAMPLE: GSM2630772\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630772\n",
            "06-Jan-2023 17:48:23 DEBUG GEOparse - SAMPLE: GSM2630773\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630773\n",
            "06-Jan-2023 17:48:23 DEBUG GEOparse - SAMPLE: GSM2630774\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630774\n",
            "06-Jan-2023 17:48:23 DEBUG GEOparse - SAMPLE: GSM2630775\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630775\n",
            "06-Jan-2023 17:48:23 DEBUG GEOparse - SAMPLE: GSM2630776\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630776\n",
            "06-Jan-2023 17:48:23 DEBUG GEOparse - SAMPLE: GSM2630777\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630777\n",
            "06-Jan-2023 17:48:24 DEBUG GEOparse - SAMPLE: GSM2630778\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630778\n",
            "06-Jan-2023 17:48:24 DEBUG GEOparse - SAMPLE: GSM2630779\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630779\n",
            "06-Jan-2023 17:48:24 DEBUG GEOparse - SAMPLE: GSM2630780\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630780\n",
            "06-Jan-2023 17:48:24 DEBUG GEOparse - SAMPLE: GSM2630781\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630781\n",
            "06-Jan-2023 17:48:24 DEBUG GEOparse - SAMPLE: GSM2630782\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630782\n",
            "06-Jan-2023 17:48:24 DEBUG GEOparse - SAMPLE: GSM2630783\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630783\n",
            "06-Jan-2023 17:48:25 DEBUG GEOparse - SAMPLE: GSM2630784\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630784\n",
            "06-Jan-2023 17:48:25 DEBUG GEOparse - SAMPLE: GSM2630785\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630785\n",
            "06-Jan-2023 17:48:25 DEBUG GEOparse - SAMPLE: GSM2630786\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630786\n",
            "06-Jan-2023 17:48:25 DEBUG GEOparse - SAMPLE: GSM2630787\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630787\n",
            "06-Jan-2023 17:48:25 DEBUG GEOparse - SAMPLE: GSM2630788\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630788\n",
            "06-Jan-2023 17:48:25 DEBUG GEOparse - SAMPLE: GSM2630789\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630789\n",
            "06-Jan-2023 17:48:26 DEBUG GEOparse - SAMPLE: GSM2630790\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630790\n",
            "06-Jan-2023 17:48:26 DEBUG GEOparse - SAMPLE: GSM2630791\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630791\n",
            "06-Jan-2023 17:48:26 DEBUG GEOparse - SAMPLE: GSM2630792\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630792\n",
            "06-Jan-2023 17:48:26 DEBUG GEOparse - SAMPLE: GSM2630793\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630793\n",
            "06-Jan-2023 17:48:26 DEBUG GEOparse - SAMPLE: GSM2630794\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630794\n",
            "06-Jan-2023 17:48:27 DEBUG GEOparse - SAMPLE: GSM2630795\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630795\n",
            "06-Jan-2023 17:48:27 DEBUG GEOparse - SAMPLE: GSM2630796\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630796\n",
            "06-Jan-2023 17:48:27 DEBUG GEOparse - SAMPLE: GSM2630797\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630797\n",
            "06-Jan-2023 17:48:27 DEBUG GEOparse - SAMPLE: GSM2630798\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630798\n",
            "06-Jan-2023 17:48:27 DEBUG GEOparse - SAMPLE: GSM2630799\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630799\n",
            "06-Jan-2023 17:48:27 DEBUG GEOparse - SAMPLE: GSM2630800\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630800\n",
            "06-Jan-2023 17:48:28 DEBUG GEOparse - SAMPLE: GSM2630801\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630801\n",
            "06-Jan-2023 17:48:28 DEBUG GEOparse - SAMPLE: GSM2630802\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630802\n",
            "06-Jan-2023 17:48:28 DEBUG GEOparse - SAMPLE: GSM2630803\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630803\n",
            "06-Jan-2023 17:48:28 DEBUG GEOparse - SAMPLE: GSM2630804\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630804\n",
            "06-Jan-2023 17:48:28 DEBUG GEOparse - SAMPLE: GSM2630805\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630805\n",
            "06-Jan-2023 17:48:28 DEBUG GEOparse - SAMPLE: GSM2630806\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630806\n",
            "06-Jan-2023 17:48:29 DEBUG GEOparse - SAMPLE: GSM2630807\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630807\n",
            "06-Jan-2023 17:48:29 DEBUG GEOparse - SAMPLE: GSM2630808\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630808\n",
            "06-Jan-2023 17:48:29 DEBUG GEOparse - SAMPLE: GSM2630809\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630809\n",
            "06-Jan-2023 17:48:29 DEBUG GEOparse - SAMPLE: GSM2630810\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630810\n",
            "06-Jan-2023 17:48:29 DEBUG GEOparse - SAMPLE: GSM2630811\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630811\n",
            "06-Jan-2023 17:48:29 DEBUG GEOparse - SAMPLE: GSM2630812\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630812\n",
            "06-Jan-2023 17:48:29 DEBUG GEOparse - SAMPLE: GSM2630813\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630813\n",
            "06-Jan-2023 17:48:30 DEBUG GEOparse - SAMPLE: GSM2630814\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630814\n",
            "06-Jan-2023 17:48:30 DEBUG GEOparse - SAMPLE: GSM2630815\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630815\n",
            "06-Jan-2023 17:48:30 DEBUG GEOparse - SAMPLE: GSM2630816\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630816\n",
            "06-Jan-2023 17:48:30 DEBUG GEOparse - SAMPLE: GSM2630817\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630817\n",
            "06-Jan-2023 17:48:30 DEBUG GEOparse - SAMPLE: GSM2630818\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630818\n",
            "06-Jan-2023 17:48:30 DEBUG GEOparse - SAMPLE: GSM2630819\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630819\n",
            "06-Jan-2023 17:48:31 DEBUG GEOparse - SAMPLE: GSM2630820\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630820\n",
            "06-Jan-2023 17:48:31 DEBUG GEOparse - SAMPLE: GSM2630821\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630821\n",
            "06-Jan-2023 17:48:31 DEBUG GEOparse - SAMPLE: GSM2630822\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630822\n",
            "06-Jan-2023 17:48:31 DEBUG GEOparse - SAMPLE: GSM2630823\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630823\n",
            "06-Jan-2023 17:48:31 DEBUG GEOparse - SAMPLE: GSM2630824\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630824\n",
            "06-Jan-2023 17:48:31 DEBUG GEOparse - SAMPLE: GSM2630825\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630825\n",
            "06-Jan-2023 17:48:31 DEBUG GEOparse - SAMPLE: GSM2630826\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630826\n",
            "06-Jan-2023 17:48:32 DEBUG GEOparse - SAMPLE: GSM2630827\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630827\n",
            "06-Jan-2023 17:48:32 DEBUG GEOparse - SAMPLE: GSM2630828\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630828\n",
            "06-Jan-2023 17:48:32 DEBUG GEOparse - SAMPLE: GSM2630829\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630829\n",
            "06-Jan-2023 17:48:32 DEBUG GEOparse - SAMPLE: GSM2630830\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630830\n",
            "06-Jan-2023 17:48:32 DEBUG GEOparse - SAMPLE: GSM2630831\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630831\n",
            "06-Jan-2023 17:48:32 DEBUG GEOparse - SAMPLE: GSM2630832\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630832\n",
            "06-Jan-2023 17:48:32 DEBUG GEOparse - SAMPLE: GSM2630833\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630833\n",
            "06-Jan-2023 17:48:33 DEBUG GEOparse - SAMPLE: GSM2630834\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630834\n",
            "06-Jan-2023 17:48:33 DEBUG GEOparse - SAMPLE: GSM2630835\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630835\n",
            "06-Jan-2023 17:48:33 DEBUG GEOparse - SAMPLE: GSM2630836\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630836\n",
            "06-Jan-2023 17:48:33 DEBUG GEOparse - SAMPLE: GSM2630837\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630837\n",
            "06-Jan-2023 17:48:33 DEBUG GEOparse - SAMPLE: GSM2630838\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630838\n",
            "06-Jan-2023 17:48:33 DEBUG GEOparse - SAMPLE: GSM2630839\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630839\n",
            "06-Jan-2023 17:48:34 DEBUG GEOparse - SAMPLE: GSM2630840\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630840\n",
            "06-Jan-2023 17:48:34 DEBUG GEOparse - SAMPLE: GSM2630841\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630841\n",
            "06-Jan-2023 17:48:34 DEBUG GEOparse - SAMPLE: GSM2630842\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630842\n",
            "06-Jan-2023 17:48:34 DEBUG GEOparse - SAMPLE: GSM2630843\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630843\n",
            "06-Jan-2023 17:48:34 DEBUG GEOparse - SAMPLE: GSM2630844\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630844\n",
            "06-Jan-2023 17:48:34 DEBUG GEOparse - SAMPLE: GSM2630845\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630845\n",
            "06-Jan-2023 17:48:35 DEBUG GEOparse - SAMPLE: GSM2630846\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630846\n",
            "06-Jan-2023 17:48:35 DEBUG GEOparse - SAMPLE: GSM2630847\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630847\n",
            "06-Jan-2023 17:48:35 DEBUG GEOparse - SAMPLE: GSM2630848\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630848\n",
            "06-Jan-2023 17:48:35 DEBUG GEOparse - SAMPLE: GSM2630849\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630849\n",
            "06-Jan-2023 17:48:35 DEBUG GEOparse - SAMPLE: GSM2630850\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630850\n",
            "06-Jan-2023 17:48:35 DEBUG GEOparse - SAMPLE: GSM2630851\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630851\n",
            "06-Jan-2023 17:48:35 DEBUG GEOparse - SAMPLE: GSM2630852\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630852\n",
            "06-Jan-2023 17:48:36 DEBUG GEOparse - SAMPLE: GSM2630853\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630853\n",
            "06-Jan-2023 17:48:36 DEBUG GEOparse - SAMPLE: GSM2630854\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630854\n",
            "06-Jan-2023 17:48:36 DEBUG GEOparse - SAMPLE: GSM2630855\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630855\n",
            "06-Jan-2023 17:48:36 DEBUG GEOparse - SAMPLE: GSM2630856\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630856\n",
            "06-Jan-2023 17:48:36 DEBUG GEOparse - SAMPLE: GSM2630857\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630857\n",
            "06-Jan-2023 17:48:36 DEBUG GEOparse - SAMPLE: GSM2630858\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630858\n",
            "06-Jan-2023 17:48:36 DEBUG GEOparse - SAMPLE: GSM2630859\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630859\n",
            "06-Jan-2023 17:48:37 DEBUG GEOparse - SAMPLE: GSM2630860\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630860\n",
            "06-Jan-2023 17:48:37 DEBUG GEOparse - SAMPLE: GSM2630861\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630861\n",
            "06-Jan-2023 17:48:37 DEBUG GEOparse - SAMPLE: GSM2630862\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630862\n",
            "06-Jan-2023 17:48:37 DEBUG GEOparse - SAMPLE: GSM2630863\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630863\n",
            "06-Jan-2023 17:48:37 DEBUG GEOparse - SAMPLE: GSM2630864\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630864\n",
            "06-Jan-2023 17:48:37 DEBUG GEOparse - SAMPLE: GSM2630865\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630865\n",
            "06-Jan-2023 17:48:37 DEBUG GEOparse - SAMPLE: GSM2630866\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630866\n",
            "06-Jan-2023 17:48:38 DEBUG GEOparse - SAMPLE: GSM2630867\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630867\n",
            "06-Jan-2023 17:48:38 DEBUG GEOparse - SAMPLE: GSM2630868\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630868\n",
            "06-Jan-2023 17:48:38 DEBUG GEOparse - SAMPLE: GSM2630869\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630869\n",
            "06-Jan-2023 17:48:38 DEBUG GEOparse - SAMPLE: GSM2630870\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630870\n",
            "06-Jan-2023 17:48:38 DEBUG GEOparse - SAMPLE: GSM2630871\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630871\n",
            "06-Jan-2023 17:48:38 DEBUG GEOparse - SAMPLE: GSM2630872\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630872\n",
            "06-Jan-2023 17:48:39 DEBUG GEOparse - SAMPLE: GSM2630873\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630873\n",
            "06-Jan-2023 17:48:39 DEBUG GEOparse - SAMPLE: GSM2630874\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630874\n",
            "06-Jan-2023 17:48:39 DEBUG GEOparse - SAMPLE: GSM2630875\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630875\n",
            "06-Jan-2023 17:48:39 DEBUG GEOparse - SAMPLE: GSM2630876\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630876\n",
            "06-Jan-2023 17:48:39 DEBUG GEOparse - SAMPLE: GSM2630877\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630877\n",
            "06-Jan-2023 17:48:39 DEBUG GEOparse - SAMPLE: GSM2630878\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630878\n",
            "06-Jan-2023 17:48:39 DEBUG GEOparse - SAMPLE: GSM2630879\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630879\n",
            "06-Jan-2023 17:48:40 DEBUG GEOparse - SAMPLE: GSM2630880\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630880\n",
            "06-Jan-2023 17:48:40 DEBUG GEOparse - SAMPLE: GSM2630881\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630881\n",
            "06-Jan-2023 17:48:40 DEBUG GEOparse - SAMPLE: GSM2630882\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630882\n",
            "06-Jan-2023 17:48:40 DEBUG GEOparse - SAMPLE: GSM2630883\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630883\n",
            "06-Jan-2023 17:48:40 DEBUG GEOparse - SAMPLE: GSM2630884\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630884\n",
            "06-Jan-2023 17:48:40 DEBUG GEOparse - SAMPLE: GSM2630885\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630885\n",
            "06-Jan-2023 17:48:41 DEBUG GEOparse - SAMPLE: GSM2630886\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630886\n",
            "06-Jan-2023 17:48:41 DEBUG GEOparse - SAMPLE: GSM2630887\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630887\n",
            "06-Jan-2023 17:48:41 DEBUG GEOparse - SAMPLE: GSM2630888\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630888\n",
            "06-Jan-2023 17:48:41 DEBUG GEOparse - SAMPLE: GSM2630889\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630889\n",
            "06-Jan-2023 17:48:41 DEBUG GEOparse - SAMPLE: GSM2630890\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630890\n",
            "06-Jan-2023 17:48:41 DEBUG GEOparse - SAMPLE: GSM2630891\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630891\n",
            "06-Jan-2023 17:48:42 DEBUG GEOparse - SAMPLE: GSM2630892\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630892\n",
            "06-Jan-2023 17:48:42 DEBUG GEOparse - SAMPLE: GSM2630893\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630893\n",
            "06-Jan-2023 17:48:42 DEBUG GEOparse - SAMPLE: GSM2630894\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630894\n",
            "06-Jan-2023 17:48:42 DEBUG GEOparse - SAMPLE: GSM2630895\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630895\n",
            "06-Jan-2023 17:48:42 DEBUG GEOparse - SAMPLE: GSM2630896\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630896\n",
            "06-Jan-2023 17:48:42 DEBUG GEOparse - SAMPLE: GSM2630897\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630897\n",
            "06-Jan-2023 17:48:42 DEBUG GEOparse - SAMPLE: GSM2630898\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630898\n",
            "06-Jan-2023 17:48:43 DEBUG GEOparse - SAMPLE: GSM2630899\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630899\n",
            "06-Jan-2023 17:48:43 DEBUG GEOparse - SAMPLE: GSM2630900\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630900\n",
            "06-Jan-2023 17:48:43 DEBUG GEOparse - SAMPLE: GSM2630901\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630901\n",
            "06-Jan-2023 17:48:43 DEBUG GEOparse - SAMPLE: GSM2630902\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630902\n",
            "06-Jan-2023 17:48:43 DEBUG GEOparse - SAMPLE: GSM2630903\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630903\n",
            "06-Jan-2023 17:48:43 DEBUG GEOparse - SAMPLE: GSM2630904\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630904\n",
            "06-Jan-2023 17:48:43 DEBUG GEOparse - SAMPLE: GSM2630905\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630905\n",
            "06-Jan-2023 17:48:44 DEBUG GEOparse - SAMPLE: GSM2630906\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630906\n",
            "06-Jan-2023 17:48:44 DEBUG GEOparse - SAMPLE: GSM2630907\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630907\n",
            "06-Jan-2023 17:48:44 DEBUG GEOparse - SAMPLE: GSM2630908\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630908\n",
            "06-Jan-2023 17:48:44 DEBUG GEOparse - SAMPLE: GSM2630909\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630909\n",
            "06-Jan-2023 17:48:44 DEBUG GEOparse - SAMPLE: GSM2630910\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630910\n",
            "06-Jan-2023 17:48:44 DEBUG GEOparse - SAMPLE: GSM2630911\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630911\n",
            "06-Jan-2023 17:48:45 DEBUG GEOparse - SAMPLE: GSM2630912\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630912\n",
            "06-Jan-2023 17:48:45 DEBUG GEOparse - SAMPLE: GSM2630913\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630913\n",
            "06-Jan-2023 17:48:45 DEBUG GEOparse - SAMPLE: GSM2630914\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630914\n",
            "06-Jan-2023 17:48:45 DEBUG GEOparse - SAMPLE: GSM2630915\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630915\n",
            "06-Jan-2023 17:48:45 DEBUG GEOparse - SAMPLE: GSM2630916\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630916\n",
            "06-Jan-2023 17:48:45 DEBUG GEOparse - SAMPLE: GSM2630917\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630917\n",
            "06-Jan-2023 17:48:45 DEBUG GEOparse - SAMPLE: GSM2630918\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630918\n",
            "06-Jan-2023 17:48:46 DEBUG GEOparse - SAMPLE: GSM2630919\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630919\n",
            "06-Jan-2023 17:48:46 DEBUG GEOparse - SAMPLE: GSM2630920\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630920\n",
            "06-Jan-2023 17:48:46 DEBUG GEOparse - SAMPLE: GSM2630921\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630921\n",
            "06-Jan-2023 17:48:46 DEBUG GEOparse - SAMPLE: GSM2630922\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630922\n",
            "06-Jan-2023 17:48:46 DEBUG GEOparse - SAMPLE: GSM2630923\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630923\n",
            "06-Jan-2023 17:48:46 DEBUG GEOparse - SAMPLE: GSM2630924\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630924\n",
            "06-Jan-2023 17:48:46 DEBUG GEOparse - SAMPLE: GSM2630925\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630925\n",
            "06-Jan-2023 17:48:47 DEBUG GEOparse - SAMPLE: GSM2630926\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630926\n",
            "06-Jan-2023 17:48:47 DEBUG GEOparse - SAMPLE: GSM2630927\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630927\n",
            "06-Jan-2023 17:48:47 DEBUG GEOparse - SAMPLE: GSM2630928\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630928\n",
            "06-Jan-2023 17:48:47 DEBUG GEOparse - SAMPLE: GSM2630929\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630929\n",
            "06-Jan-2023 17:48:47 DEBUG GEOparse - SAMPLE: GSM2630930\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630930\n",
            "06-Jan-2023 17:48:47 DEBUG GEOparse - SAMPLE: GSM2630931\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630931\n",
            "06-Jan-2023 17:48:48 DEBUG GEOparse - SAMPLE: GSM2630932\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630932\n",
            "06-Jan-2023 17:48:48 DEBUG GEOparse - SAMPLE: GSM2630933\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630933\n",
            "06-Jan-2023 17:48:48 DEBUG GEOparse - SAMPLE: GSM2630934\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630934\n",
            "06-Jan-2023 17:48:48 DEBUG GEOparse - SAMPLE: GSM2630935\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630935\n",
            "06-Jan-2023 17:48:48 DEBUG GEOparse - SAMPLE: GSM2630936\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630936\n",
            "06-Jan-2023 17:48:48 DEBUG GEOparse - SAMPLE: GSM2630937\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630937\n",
            "06-Jan-2023 17:48:49 DEBUG GEOparse - SAMPLE: GSM2630938\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630938\n",
            "06-Jan-2023 17:48:49 DEBUG GEOparse - SAMPLE: GSM2630939\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630939\n",
            "06-Jan-2023 17:48:49 DEBUG GEOparse - SAMPLE: GSM2630940\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630940\n",
            "06-Jan-2023 17:48:49 DEBUG GEOparse - SAMPLE: GSM2630941\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630941\n",
            "06-Jan-2023 17:48:49 DEBUG GEOparse - SAMPLE: GSM2630942\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630942\n",
            "06-Jan-2023 17:48:49 DEBUG GEOparse - SAMPLE: GSM2630943\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630943\n",
            "06-Jan-2023 17:48:50 DEBUG GEOparse - SAMPLE: GSM2630944\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630944\n",
            "06-Jan-2023 17:48:50 DEBUG GEOparse - SAMPLE: GSM2630945\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630945\n",
            "06-Jan-2023 17:48:50 DEBUG GEOparse - SAMPLE: GSM2630946\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630946\n",
            "06-Jan-2023 17:48:50 DEBUG GEOparse - SAMPLE: GSM2630947\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630947\n",
            "06-Jan-2023 17:48:50 DEBUG GEOparse - SAMPLE: GSM2630948\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630948\n",
            "06-Jan-2023 17:48:50 DEBUG GEOparse - SAMPLE: GSM2630949\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630949\n",
            "06-Jan-2023 17:48:50 DEBUG GEOparse - SAMPLE: GSM2630950\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630950\n",
            "06-Jan-2023 17:48:51 DEBUG GEOparse - SAMPLE: GSM2630951\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630951\n",
            "06-Jan-2023 17:48:51 DEBUG GEOparse - SAMPLE: GSM2630952\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630952\n",
            "06-Jan-2023 17:48:51 DEBUG GEOparse - SAMPLE: GSM2630953\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630953\n",
            "06-Jan-2023 17:48:51 DEBUG GEOparse - SAMPLE: GSM2630954\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630954\n",
            "06-Jan-2023 17:48:51 DEBUG GEOparse - SAMPLE: GSM2630955\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630955\n",
            "06-Jan-2023 17:48:51 DEBUG GEOparse - SAMPLE: GSM2630956\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630956\n",
            "06-Jan-2023 17:48:52 DEBUG GEOparse - SAMPLE: GSM2630957\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630957\n",
            "06-Jan-2023 17:48:52 DEBUG GEOparse - SAMPLE: GSM2630958\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630958\n",
            "06-Jan-2023 17:48:52 DEBUG GEOparse - SAMPLE: GSM2630959\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630959\n",
            "06-Jan-2023 17:48:52 DEBUG GEOparse - SAMPLE: GSM2630960\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630960\n",
            "06-Jan-2023 17:48:52 DEBUG GEOparse - SAMPLE: GSM2630961\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630961\n",
            "06-Jan-2023 17:48:52 DEBUG GEOparse - SAMPLE: GSM2630962\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630962\n",
            "06-Jan-2023 17:48:52 DEBUG GEOparse - SAMPLE: GSM2630963\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630963\n",
            "06-Jan-2023 17:48:53 DEBUG GEOparse - SAMPLE: GSM2630964\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630964\n",
            "06-Jan-2023 17:48:53 DEBUG GEOparse - SAMPLE: GSM2630965\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630965\n",
            "06-Jan-2023 17:48:53 DEBUG GEOparse - SAMPLE: GSM2630966\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630966\n",
            "06-Jan-2023 17:48:53 DEBUG GEOparse - SAMPLE: GSM2630967\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630967\n",
            "06-Jan-2023 17:48:53 DEBUG GEOparse - SAMPLE: GSM2630968\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630968\n",
            "06-Jan-2023 17:48:53 DEBUG GEOparse - SAMPLE: GSM2630969\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630969\n",
            "06-Jan-2023 17:48:54 DEBUG GEOparse - SAMPLE: GSM2630970\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630970\n",
            "06-Jan-2023 17:48:54 DEBUG GEOparse - SAMPLE: GSM2630971\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630971\n",
            "06-Jan-2023 17:48:54 DEBUG GEOparse - SAMPLE: GSM2630972\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630972\n",
            "06-Jan-2023 17:48:54 DEBUG GEOparse - SAMPLE: GSM2630973\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630973\n",
            "06-Jan-2023 17:48:54 DEBUG GEOparse - SAMPLE: GSM2630974\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630974\n",
            "06-Jan-2023 17:48:54 DEBUG GEOparse - SAMPLE: GSM2630975\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630975\n",
            "06-Jan-2023 17:48:55 DEBUG GEOparse - SAMPLE: GSM2630976\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630976\n",
            "06-Jan-2023 17:48:55 DEBUG GEOparse - SAMPLE: GSM2630977\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630977\n",
            "06-Jan-2023 17:48:55 DEBUG GEOparse - SAMPLE: GSM2630978\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630978\n",
            "06-Jan-2023 17:48:55 DEBUG GEOparse - SAMPLE: GSM2630979\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630979\n",
            "06-Jan-2023 17:48:55 DEBUG GEOparse - SAMPLE: GSM2630980\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630980\n",
            "06-Jan-2023 17:48:55 DEBUG GEOparse - SAMPLE: GSM2630981\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630981\n",
            "06-Jan-2023 17:48:56 DEBUG GEOparse - SAMPLE: GSM2630982\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630982\n",
            "06-Jan-2023 17:48:56 DEBUG GEOparse - SAMPLE: GSM2630983\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630983\n",
            "06-Jan-2023 17:48:56 DEBUG GEOparse - SAMPLE: GSM2630984\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630984\n",
            "06-Jan-2023 17:48:56 DEBUG GEOparse - SAMPLE: GSM2630985\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630985\n",
            "06-Jan-2023 17:48:56 DEBUG GEOparse - SAMPLE: GSM2630986\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630986\n",
            "06-Jan-2023 17:48:56 DEBUG GEOparse - SAMPLE: GSM2630987\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630987\n",
            "06-Jan-2023 17:48:56 DEBUG GEOparse - SAMPLE: GSM2630988\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630988\n",
            "06-Jan-2023 17:48:57 DEBUG GEOparse - SAMPLE: GSM2630989\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630989\n",
            "06-Jan-2023 17:48:57 DEBUG GEOparse - SAMPLE: GSM2630990\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630990\n",
            "06-Jan-2023 17:48:57 DEBUG GEOparse - SAMPLE: GSM2630991\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630991\n",
            "06-Jan-2023 17:48:57 DEBUG GEOparse - SAMPLE: GSM2630992\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630992\n",
            "06-Jan-2023 17:48:57 DEBUG GEOparse - SAMPLE: GSM2630993\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630993\n",
            "06-Jan-2023 17:48:57 DEBUG GEOparse - SAMPLE: GSM2630994\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630994\n",
            "06-Jan-2023 17:48:58 DEBUG GEOparse - SAMPLE: GSM2630995\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630995\n",
            "06-Jan-2023 17:48:58 DEBUG GEOparse - SAMPLE: GSM2630996\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630996\n",
            "06-Jan-2023 17:48:58 DEBUG GEOparse - SAMPLE: GSM2630997\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630997\n",
            "06-Jan-2023 17:48:58 DEBUG GEOparse - SAMPLE: GSM2630998\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630998\n",
            "06-Jan-2023 17:48:58 DEBUG GEOparse - SAMPLE: GSM2630999\n",
            "DEBUG:GEOparse:SAMPLE: GSM2630999\n",
            "06-Jan-2023 17:48:58 DEBUG GEOparse - SAMPLE: GSM2631000\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631000\n",
            "06-Jan-2023 17:48:59 DEBUG GEOparse - SAMPLE: GSM2631001\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631001\n",
            "06-Jan-2023 17:48:59 DEBUG GEOparse - SAMPLE: GSM2631002\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631002\n",
            "06-Jan-2023 17:48:59 DEBUG GEOparse - SAMPLE: GSM2631003\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631003\n",
            "06-Jan-2023 17:48:59 DEBUG GEOparse - SAMPLE: GSM2631004\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631004\n",
            "06-Jan-2023 17:48:59 DEBUG GEOparse - SAMPLE: GSM2631005\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631005\n",
            "06-Jan-2023 17:48:59 DEBUG GEOparse - SAMPLE: GSM2631006\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631006\n",
            "06-Jan-2023 17:49:00 DEBUG GEOparse - SAMPLE: GSM2631007\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631007\n",
            "06-Jan-2023 17:49:00 DEBUG GEOparse - SAMPLE: GSM2631008\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631008\n",
            "06-Jan-2023 17:49:00 DEBUG GEOparse - SAMPLE: GSM2631009\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631009\n",
            "06-Jan-2023 17:49:00 DEBUG GEOparse - SAMPLE: GSM2631010\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631010\n",
            "06-Jan-2023 17:49:00 DEBUG GEOparse - SAMPLE: GSM2631011\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631011\n",
            "06-Jan-2023 17:49:00 DEBUG GEOparse - SAMPLE: GSM2631012\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631012\n",
            "06-Jan-2023 17:49:00 DEBUG GEOparse - SAMPLE: GSM2631013\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631013\n",
            "06-Jan-2023 17:49:01 DEBUG GEOparse - SAMPLE: GSM2631014\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631014\n",
            "06-Jan-2023 17:49:01 DEBUG GEOparse - SAMPLE: GSM2631015\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631015\n",
            "06-Jan-2023 17:49:01 DEBUG GEOparse - SAMPLE: GSM2631016\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631016\n",
            "06-Jan-2023 17:49:01 DEBUG GEOparse - SAMPLE: GSM2631017\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631017\n",
            "06-Jan-2023 17:49:01 DEBUG GEOparse - SAMPLE: GSM2631018\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631018\n",
            "06-Jan-2023 17:49:01 DEBUG GEOparse - SAMPLE: GSM2631019\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631019\n",
            "06-Jan-2023 17:49:02 DEBUG GEOparse - SAMPLE: GSM2631020\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631020\n",
            "06-Jan-2023 17:49:02 DEBUG GEOparse - SAMPLE: GSM2631021\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631021\n",
            "06-Jan-2023 17:49:02 DEBUG GEOparse - SAMPLE: GSM2631022\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631022\n",
            "06-Jan-2023 17:49:02 DEBUG GEOparse - SAMPLE: GSM2631023\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631023\n",
            "06-Jan-2023 17:49:02 DEBUG GEOparse - SAMPLE: GSM2631024\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631024\n",
            "06-Jan-2023 17:49:02 DEBUG GEOparse - SAMPLE: GSM2631025\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631025\n",
            "06-Jan-2023 17:49:03 DEBUG GEOparse - SAMPLE: GSM2631026\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631026\n",
            "06-Jan-2023 17:49:03 DEBUG GEOparse - SAMPLE: GSM2631027\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631027\n",
            "06-Jan-2023 17:49:03 DEBUG GEOparse - SAMPLE: GSM2631028\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631028\n",
            "06-Jan-2023 17:49:03 DEBUG GEOparse - SAMPLE: GSM2631029\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631029\n",
            "06-Jan-2023 17:49:03 DEBUG GEOparse - SAMPLE: GSM2631030\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631030\n",
            "06-Jan-2023 17:49:03 DEBUG GEOparse - SAMPLE: GSM2631031\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631031\n",
            "06-Jan-2023 17:49:04 DEBUG GEOparse - SAMPLE: GSM2631032\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631032\n",
            "06-Jan-2023 17:49:04 DEBUG GEOparse - SAMPLE: GSM2631033\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631033\n",
            "06-Jan-2023 17:49:04 DEBUG GEOparse - SAMPLE: GSM2631034\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631034\n",
            "06-Jan-2023 17:49:04 DEBUG GEOparse - SAMPLE: GSM2631035\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631035\n",
            "06-Jan-2023 17:49:04 DEBUG GEOparse - SAMPLE: GSM2631036\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631036\n",
            "06-Jan-2023 17:49:04 DEBUG GEOparse - SAMPLE: GSM2631037\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631037\n",
            "06-Jan-2023 17:49:05 DEBUG GEOparse - SAMPLE: GSM2631038\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631038\n",
            "06-Jan-2023 17:49:05 DEBUG GEOparse - SAMPLE: GSM2631039\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631039\n",
            "06-Jan-2023 17:49:05 DEBUG GEOparse - SAMPLE: GSM2631040\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631040\n",
            "06-Jan-2023 17:49:05 DEBUG GEOparse - SAMPLE: GSM2631041\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631041\n",
            "06-Jan-2023 17:49:05 DEBUG GEOparse - SAMPLE: GSM2631042\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631042\n",
            "06-Jan-2023 17:49:05 DEBUG GEOparse - SAMPLE: GSM2631043\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631043\n",
            "06-Jan-2023 17:49:06 DEBUG GEOparse - SAMPLE: GSM2631044\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631044\n",
            "06-Jan-2023 17:49:06 DEBUG GEOparse - SAMPLE: GSM2631045\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631045\n",
            "06-Jan-2023 17:49:06 DEBUG GEOparse - SAMPLE: GSM2631046\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631046\n",
            "06-Jan-2023 17:49:06 DEBUG GEOparse - SAMPLE: GSM2631047\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631047\n",
            "06-Jan-2023 17:49:06 DEBUG GEOparse - SAMPLE: GSM2631048\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631048\n",
            "06-Jan-2023 17:49:06 DEBUG GEOparse - SAMPLE: GSM2631049\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631049\n",
            "06-Jan-2023 17:49:07 DEBUG GEOparse - SAMPLE: GSM2631050\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631050\n",
            "06-Jan-2023 17:49:07 DEBUG GEOparse - SAMPLE: GSM2631051\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631051\n",
            "06-Jan-2023 17:49:07 DEBUG GEOparse - SAMPLE: GSM2631052\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631052\n",
            "06-Jan-2023 17:49:07 DEBUG GEOparse - SAMPLE: GSM2631053\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631053\n",
            "06-Jan-2023 17:49:07 DEBUG GEOparse - SAMPLE: GSM2631054\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631054\n",
            "06-Jan-2023 17:49:07 DEBUG GEOparse - SAMPLE: GSM2631055\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631055\n",
            "06-Jan-2023 17:49:08 DEBUG GEOparse - SAMPLE: GSM2631056\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631056\n",
            "06-Jan-2023 17:49:08 DEBUG GEOparse - SAMPLE: GSM2631057\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631057\n",
            "06-Jan-2023 17:49:08 DEBUG GEOparse - SAMPLE: GSM2631058\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631058\n",
            "06-Jan-2023 17:49:08 DEBUG GEOparse - SAMPLE: GSM2631059\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631059\n",
            "06-Jan-2023 17:49:08 DEBUG GEOparse - SAMPLE: GSM2631060\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631060\n",
            "06-Jan-2023 17:49:08 DEBUG GEOparse - SAMPLE: GSM2631061\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631061\n",
            "06-Jan-2023 17:49:09 DEBUG GEOparse - SAMPLE: GSM2631062\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631062\n",
            "06-Jan-2023 17:49:09 DEBUG GEOparse - SAMPLE: GSM2631063\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631063\n",
            "06-Jan-2023 17:49:09 DEBUG GEOparse - SAMPLE: GSM2631064\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631064\n",
            "06-Jan-2023 17:49:09 DEBUG GEOparse - SAMPLE: GSM2631065\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631065\n",
            "06-Jan-2023 17:49:09 DEBUG GEOparse - SAMPLE: GSM2631066\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631066\n",
            "06-Jan-2023 17:49:09 DEBUG GEOparse - SAMPLE: GSM2631067\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631067\n",
            "06-Jan-2023 17:49:10 DEBUG GEOparse - SAMPLE: GSM2631068\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631068\n",
            "06-Jan-2023 17:49:10 DEBUG GEOparse - SAMPLE: GSM2631069\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631069\n",
            "06-Jan-2023 17:49:10 DEBUG GEOparse - SAMPLE: GSM2631070\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631070\n",
            "06-Jan-2023 17:49:10 DEBUG GEOparse - SAMPLE: GSM2631071\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631071\n",
            "06-Jan-2023 17:49:10 DEBUG GEOparse - SAMPLE: GSM2631072\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631072\n",
            "06-Jan-2023 17:49:10 DEBUG GEOparse - SAMPLE: GSM2631073\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631073\n",
            "06-Jan-2023 17:49:11 DEBUG GEOparse - SAMPLE: GSM2631074\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631074\n",
            "06-Jan-2023 17:49:11 DEBUG GEOparse - SAMPLE: GSM2631075\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631075\n",
            "06-Jan-2023 17:49:11 DEBUG GEOparse - SAMPLE: GSM2631076\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631076\n",
            "06-Jan-2023 17:49:11 DEBUG GEOparse - SAMPLE: GSM2631077\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631077\n",
            "06-Jan-2023 17:49:11 DEBUG GEOparse - SAMPLE: GSM2631078\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631078\n",
            "06-Jan-2023 17:49:11 DEBUG GEOparse - SAMPLE: GSM2631079\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631079\n",
            "06-Jan-2023 17:49:12 DEBUG GEOparse - SAMPLE: GSM2631080\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631080\n",
            "06-Jan-2023 17:49:12 DEBUG GEOparse - SAMPLE: GSM2631081\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631081\n",
            "06-Jan-2023 17:49:12 DEBUG GEOparse - SAMPLE: GSM2631082\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631082\n",
            "06-Jan-2023 17:49:12 DEBUG GEOparse - SAMPLE: GSM2631083\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631083\n",
            "06-Jan-2023 17:49:12 DEBUG GEOparse - SAMPLE: GSM2631084\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631084\n",
            "06-Jan-2023 17:49:12 DEBUG GEOparse - SAMPLE: GSM2631085\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631085\n",
            "06-Jan-2023 17:49:13 DEBUG GEOparse - SAMPLE: GSM2631086\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631086\n",
            "06-Jan-2023 17:49:13 DEBUG GEOparse - SAMPLE: GSM2631087\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631087\n",
            "06-Jan-2023 17:49:13 DEBUG GEOparse - SAMPLE: GSM2631088\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631088\n",
            "06-Jan-2023 17:49:13 DEBUG GEOparse - SAMPLE: GSM2631089\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631089\n",
            "06-Jan-2023 17:49:13 DEBUG GEOparse - SAMPLE: GSM2631090\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631090\n",
            "06-Jan-2023 17:49:13 DEBUG GEOparse - SAMPLE: GSM2631091\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631091\n",
            "06-Jan-2023 17:49:13 DEBUG GEOparse - SAMPLE: GSM2631092\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631092\n",
            "06-Jan-2023 17:49:14 DEBUG GEOparse - SAMPLE: GSM2631093\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631093\n",
            "06-Jan-2023 17:49:14 DEBUG GEOparse - SAMPLE: GSM2631094\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631094\n",
            "06-Jan-2023 17:49:14 DEBUG GEOparse - SAMPLE: GSM2631095\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631095\n",
            "06-Jan-2023 17:49:14 DEBUG GEOparse - SAMPLE: GSM2631096\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631096\n",
            "06-Jan-2023 17:49:14 DEBUG GEOparse - SAMPLE: GSM2631097\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631097\n",
            "06-Jan-2023 17:49:14 DEBUG GEOparse - SAMPLE: GSM2631098\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631098\n",
            "06-Jan-2023 17:49:15 DEBUG GEOparse - SAMPLE: GSM2631099\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631099\n",
            "06-Jan-2023 17:49:15 DEBUG GEOparse - SAMPLE: GSM2631100\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631100\n",
            "06-Jan-2023 17:49:15 DEBUG GEOparse - SAMPLE: GSM2631101\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631101\n",
            "06-Jan-2023 17:49:15 DEBUG GEOparse - SAMPLE: GSM2631102\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631102\n",
            "06-Jan-2023 17:49:15 DEBUG GEOparse - SAMPLE: GSM2631103\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631103\n",
            "06-Jan-2023 17:49:15 DEBUG GEOparse - SAMPLE: GSM2631104\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631104\n",
            "06-Jan-2023 17:49:16 DEBUG GEOparse - SAMPLE: GSM2631105\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631105\n",
            "06-Jan-2023 17:49:16 DEBUG GEOparse - SAMPLE: GSM2631106\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631106\n",
            "06-Jan-2023 17:49:16 DEBUG GEOparse - SAMPLE: GSM2631107\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631107\n",
            "06-Jan-2023 17:49:16 DEBUG GEOparse - SAMPLE: GSM2631108\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631108\n",
            "06-Jan-2023 17:49:16 DEBUG GEOparse - SAMPLE: GSM2631109\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631109\n",
            "06-Jan-2023 17:49:16 DEBUG GEOparse - SAMPLE: GSM2631110\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631110\n",
            "06-Jan-2023 17:49:17 DEBUG GEOparse - SAMPLE: GSM2631111\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631111\n",
            "06-Jan-2023 17:49:17 DEBUG GEOparse - SAMPLE: GSM2631112\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631112\n",
            "06-Jan-2023 17:49:17 DEBUG GEOparse - SAMPLE: GSM2631113\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631113\n",
            "06-Jan-2023 17:49:17 DEBUG GEOparse - SAMPLE: GSM2631114\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631114\n",
            "06-Jan-2023 17:49:17 DEBUG GEOparse - SAMPLE: GSM2631115\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631115\n",
            "06-Jan-2023 17:49:17 DEBUG GEOparse - SAMPLE: GSM2631116\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631116\n",
            "06-Jan-2023 17:49:18 DEBUG GEOparse - SAMPLE: GSM2631117\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631117\n",
            "06-Jan-2023 17:49:18 DEBUG GEOparse - SAMPLE: GSM2631118\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631118\n",
            "06-Jan-2023 17:49:18 DEBUG GEOparse - SAMPLE: GSM2631119\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631119\n",
            "06-Jan-2023 17:49:18 DEBUG GEOparse - SAMPLE: GSM2631120\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631120\n",
            "06-Jan-2023 17:49:18 DEBUG GEOparse - SAMPLE: GSM2631121\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631121\n",
            "06-Jan-2023 17:49:18 DEBUG GEOparse - SAMPLE: GSM2631122\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631122\n",
            "06-Jan-2023 17:49:19 DEBUG GEOparse - SAMPLE: GSM2631123\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631123\n",
            "06-Jan-2023 17:49:19 DEBUG GEOparse - SAMPLE: GSM2631124\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631124\n",
            "06-Jan-2023 17:49:19 DEBUG GEOparse - SAMPLE: GSM2631125\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631125\n",
            "06-Jan-2023 17:49:19 DEBUG GEOparse - SAMPLE: GSM2631126\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631126\n",
            "06-Jan-2023 17:49:19 DEBUG GEOparse - SAMPLE: GSM2631127\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631127\n",
            "06-Jan-2023 17:49:19 DEBUG GEOparse - SAMPLE: GSM2631128\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631128\n",
            "06-Jan-2023 17:49:20 DEBUG GEOparse - SAMPLE: GSM2631129\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631129\n",
            "06-Jan-2023 17:49:20 DEBUG GEOparse - SAMPLE: GSM2631130\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631130\n",
            "06-Jan-2023 17:49:20 DEBUG GEOparse - SAMPLE: GSM2631131\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631131\n",
            "06-Jan-2023 17:49:20 DEBUG GEOparse - SAMPLE: GSM2631132\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631132\n",
            "06-Jan-2023 17:49:20 DEBUG GEOparse - SAMPLE: GSM2631133\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631133\n",
            "06-Jan-2023 17:49:20 DEBUG GEOparse - SAMPLE: GSM2631134\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631134\n",
            "06-Jan-2023 17:49:20 DEBUG GEOparse - SAMPLE: GSM2631135\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631135\n",
            "06-Jan-2023 17:49:21 DEBUG GEOparse - SAMPLE: GSM2631136\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631136\n",
            "06-Jan-2023 17:49:21 DEBUG GEOparse - SAMPLE: GSM2631137\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631137\n",
            "06-Jan-2023 17:49:21 DEBUG GEOparse - SAMPLE: GSM2631138\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631138\n",
            "06-Jan-2023 17:49:21 DEBUG GEOparse - SAMPLE: GSM2631139\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631139\n",
            "06-Jan-2023 17:49:21 DEBUG GEOparse - SAMPLE: GSM2631140\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631140\n",
            "06-Jan-2023 17:49:21 DEBUG GEOparse - SAMPLE: GSM2631141\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631141\n",
            "06-Jan-2023 17:49:22 DEBUG GEOparse - SAMPLE: GSM2631142\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631142\n",
            "06-Jan-2023 17:49:22 DEBUG GEOparse - SAMPLE: GSM2631143\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631143\n",
            "06-Jan-2023 17:49:22 DEBUG GEOparse - SAMPLE: GSM2631144\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631144\n",
            "06-Jan-2023 17:49:22 DEBUG GEOparse - SAMPLE: GSM2631145\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631145\n",
            "06-Jan-2023 17:49:22 DEBUG GEOparse - SAMPLE: GSM2631146\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631146\n",
            "06-Jan-2023 17:49:22 DEBUG GEOparse - SAMPLE: GSM2631147\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631147\n",
            "06-Jan-2023 17:49:22 DEBUG GEOparse - SAMPLE: GSM2631148\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631148\n",
            "06-Jan-2023 17:49:23 DEBUG GEOparse - SAMPLE: GSM2631149\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631149\n",
            "06-Jan-2023 17:49:23 DEBUG GEOparse - SAMPLE: GSM2631150\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631150\n",
            "06-Jan-2023 17:49:23 DEBUG GEOparse - SAMPLE: GSM2631151\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631151\n",
            "06-Jan-2023 17:49:23 DEBUG GEOparse - SAMPLE: GSM2631152\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631152\n",
            "06-Jan-2023 17:49:23 DEBUG GEOparse - SAMPLE: GSM2631153\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631153\n",
            "06-Jan-2023 17:49:23 DEBUG GEOparse - SAMPLE: GSM2631154\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631154\n",
            "06-Jan-2023 17:49:24 DEBUG GEOparse - SAMPLE: GSM2631155\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631155\n",
            "06-Jan-2023 17:49:24 DEBUG GEOparse - SAMPLE: GSM2631156\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631156\n",
            "06-Jan-2023 17:49:24 DEBUG GEOparse - SAMPLE: GSM2631157\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631157\n",
            "06-Jan-2023 17:49:24 DEBUG GEOparse - SAMPLE: GSM2631158\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631158\n",
            "06-Jan-2023 17:49:24 DEBUG GEOparse - SAMPLE: GSM2631159\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631159\n",
            "06-Jan-2023 17:49:25 DEBUG GEOparse - SAMPLE: GSM2631160\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631160\n",
            "06-Jan-2023 17:49:25 DEBUG GEOparse - SAMPLE: GSM2631161\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631161\n",
            "06-Jan-2023 17:49:25 DEBUG GEOparse - SAMPLE: GSM2631162\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631162\n",
            "06-Jan-2023 17:49:25 DEBUG GEOparse - SAMPLE: GSM2631163\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631163\n",
            "06-Jan-2023 17:49:25 DEBUG GEOparse - SAMPLE: GSM2631164\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631164\n",
            "06-Jan-2023 17:49:25 DEBUG GEOparse - SAMPLE: GSM2631165\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631165\n",
            "06-Jan-2023 17:49:26 DEBUG GEOparse - SAMPLE: GSM2631166\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631166\n",
            "06-Jan-2023 17:49:26 DEBUG GEOparse - SAMPLE: GSM2631167\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631167\n",
            "06-Jan-2023 17:49:26 DEBUG GEOparse - SAMPLE: GSM2631168\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631168\n",
            "06-Jan-2023 17:49:26 DEBUG GEOparse - SAMPLE: GSM2631169\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631169\n",
            "06-Jan-2023 17:49:26 DEBUG GEOparse - SAMPLE: GSM2631170\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631170\n",
            "06-Jan-2023 17:49:26 DEBUG GEOparse - SAMPLE: GSM2631171\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631171\n",
            "06-Jan-2023 17:49:26 DEBUG GEOparse - SAMPLE: GSM2631172\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631172\n",
            "06-Jan-2023 17:49:27 DEBUG GEOparse - SAMPLE: GSM2631173\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631173\n",
            "06-Jan-2023 17:49:27 DEBUG GEOparse - SAMPLE: GSM2631174\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631174\n",
            "06-Jan-2023 17:49:27 DEBUG GEOparse - SAMPLE: GSM2631175\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631175\n",
            "06-Jan-2023 17:49:27 DEBUG GEOparse - SAMPLE: GSM2631176\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631176\n",
            "06-Jan-2023 17:49:27 DEBUG GEOparse - SAMPLE: GSM2631177\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631177\n",
            "06-Jan-2023 17:49:27 DEBUG GEOparse - SAMPLE: GSM2631178\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631178\n",
            "06-Jan-2023 17:49:28 DEBUG GEOparse - SAMPLE: GSM2631179\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631179\n",
            "06-Jan-2023 17:49:28 DEBUG GEOparse - SAMPLE: GSM2631180\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631180\n",
            "06-Jan-2023 17:49:28 DEBUG GEOparse - SAMPLE: GSM2631181\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631181\n",
            "06-Jan-2023 17:49:28 DEBUG GEOparse - SAMPLE: GSM2631182\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631182\n",
            "06-Jan-2023 17:49:28 DEBUG GEOparse - SAMPLE: GSM2631183\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631183\n",
            "06-Jan-2023 17:49:28 DEBUG GEOparse - SAMPLE: GSM2631184\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631184\n",
            "06-Jan-2023 17:49:29 DEBUG GEOparse - SAMPLE: GSM2631185\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631185\n",
            "06-Jan-2023 17:49:29 DEBUG GEOparse - SAMPLE: GSM2631186\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631186\n",
            "06-Jan-2023 17:49:29 DEBUG GEOparse - SAMPLE: GSM2631187\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631187\n",
            "06-Jan-2023 17:49:29 DEBUG GEOparse - SAMPLE: GSM2631188\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631188\n",
            "06-Jan-2023 17:49:29 DEBUG GEOparse - SAMPLE: GSM2631189\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631189\n",
            "06-Jan-2023 17:49:29 DEBUG GEOparse - SAMPLE: GSM2631190\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631190\n",
            "06-Jan-2023 17:49:30 DEBUG GEOparse - SAMPLE: GSM2631191\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631191\n",
            "06-Jan-2023 17:49:30 DEBUG GEOparse - SAMPLE: GSM2631192\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631192\n",
            "06-Jan-2023 17:49:30 DEBUG GEOparse - SAMPLE: GSM2631193\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631193\n",
            "06-Jan-2023 17:49:30 DEBUG GEOparse - SAMPLE: GSM2631194\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631194\n",
            "06-Jan-2023 17:49:30 DEBUG GEOparse - SAMPLE: GSM2631195\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631195\n",
            "06-Jan-2023 17:49:30 DEBUG GEOparse - SAMPLE: GSM2631196\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631196\n",
            "06-Jan-2023 17:49:31 DEBUG GEOparse - SAMPLE: GSM2631197\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631197\n",
            "06-Jan-2023 17:49:31 DEBUG GEOparse - SAMPLE: GSM2631198\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631198\n",
            "06-Jan-2023 17:49:31 DEBUG GEOparse - SAMPLE: GSM2631199\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631199\n",
            "06-Jan-2023 17:49:31 DEBUG GEOparse - SAMPLE: GSM2631200\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631200\n",
            "06-Jan-2023 17:49:31 DEBUG GEOparse - SAMPLE: GSM2631201\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631201\n",
            "06-Jan-2023 17:49:31 DEBUG GEOparse - SAMPLE: GSM2631202\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631202\n",
            "06-Jan-2023 17:49:32 DEBUG GEOparse - SAMPLE: GSM2631203\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631203\n",
            "06-Jan-2023 17:49:32 DEBUG GEOparse - SAMPLE: GSM2631204\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631204\n",
            "06-Jan-2023 17:49:32 DEBUG GEOparse - SAMPLE: GSM2631205\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631205\n",
            "06-Jan-2023 17:49:32 DEBUG GEOparse - SAMPLE: GSM2631206\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631206\n",
            "06-Jan-2023 17:49:32 DEBUG GEOparse - SAMPLE: GSM2631207\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631207\n",
            "06-Jan-2023 17:49:32 DEBUG GEOparse - SAMPLE: GSM2631208\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631208\n",
            "06-Jan-2023 17:49:33 DEBUG GEOparse - SAMPLE: GSM2631209\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631209\n",
            "06-Jan-2023 17:49:33 DEBUG GEOparse - SAMPLE: GSM2631210\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631210\n",
            "06-Jan-2023 17:49:33 DEBUG GEOparse - SAMPLE: GSM2631211\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631211\n",
            "06-Jan-2023 17:49:33 DEBUG GEOparse - SAMPLE: GSM2631212\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631212\n",
            "06-Jan-2023 17:49:33 DEBUG GEOparse - SAMPLE: GSM2631213\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631213\n",
            "06-Jan-2023 17:49:33 DEBUG GEOparse - SAMPLE: GSM2631214\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631214\n",
            "06-Jan-2023 17:49:34 DEBUG GEOparse - SAMPLE: GSM2631215\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631215\n",
            "06-Jan-2023 17:49:34 DEBUG GEOparse - SAMPLE: GSM2631216\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631216\n",
            "06-Jan-2023 17:49:34 DEBUG GEOparse - SAMPLE: GSM2631217\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631217\n",
            "06-Jan-2023 17:49:34 DEBUG GEOparse - SAMPLE: GSM2631218\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631218\n",
            "06-Jan-2023 17:49:34 DEBUG GEOparse - SAMPLE: GSM2631219\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631219\n",
            "06-Jan-2023 17:49:34 DEBUG GEOparse - SAMPLE: GSM2631220\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631220\n",
            "06-Jan-2023 17:49:35 DEBUG GEOparse - SAMPLE: GSM2631221\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631221\n",
            "06-Jan-2023 17:49:35 DEBUG GEOparse - SAMPLE: GSM2631222\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631222\n",
            "06-Jan-2023 17:49:35 DEBUG GEOparse - SAMPLE: GSM2631223\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631223\n",
            "06-Jan-2023 17:49:35 DEBUG GEOparse - SAMPLE: GSM2631224\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631224\n",
            "06-Jan-2023 17:49:35 DEBUG GEOparse - SAMPLE: GSM2631225\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631225\n",
            "06-Jan-2023 17:49:35 DEBUG GEOparse - SAMPLE: GSM2631226\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631226\n",
            "06-Jan-2023 17:49:36 DEBUG GEOparse - SAMPLE: GSM2631227\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631227\n",
            "06-Jan-2023 17:49:36 DEBUG GEOparse - SAMPLE: GSM2631228\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631228\n",
            "06-Jan-2023 17:49:36 DEBUG GEOparse - SAMPLE: GSM2631229\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631229\n",
            "06-Jan-2023 17:49:36 DEBUG GEOparse - SAMPLE: GSM2631230\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631230\n",
            "06-Jan-2023 17:49:36 DEBUG GEOparse - SAMPLE: GSM2631231\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631231\n",
            "06-Jan-2023 17:49:36 DEBUG GEOparse - SAMPLE: GSM2631232\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631232\n",
            "06-Jan-2023 17:49:37 DEBUG GEOparse - SAMPLE: GSM2631233\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631233\n",
            "06-Jan-2023 17:49:37 DEBUG GEOparse - SAMPLE: GSM2631234\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631234\n",
            "06-Jan-2023 17:49:37 DEBUG GEOparse - SAMPLE: GSM2631235\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631235\n",
            "06-Jan-2023 17:49:37 DEBUG GEOparse - SAMPLE: GSM2631236\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631236\n",
            "06-Jan-2023 17:49:37 DEBUG GEOparse - SAMPLE: GSM2631237\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631237\n",
            "06-Jan-2023 17:49:37 DEBUG GEOparse - SAMPLE: GSM2631238\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631238\n",
            "06-Jan-2023 17:49:37 DEBUG GEOparse - SAMPLE: GSM2631239\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631239\n",
            "06-Jan-2023 17:49:38 DEBUG GEOparse - SAMPLE: GSM2631240\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631240\n",
            "06-Jan-2023 17:49:38 DEBUG GEOparse - SAMPLE: GSM2631241\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631241\n",
            "06-Jan-2023 17:49:38 DEBUG GEOparse - SAMPLE: GSM2631242\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631242\n",
            "06-Jan-2023 17:49:38 DEBUG GEOparse - SAMPLE: GSM2631243\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631243\n",
            "06-Jan-2023 17:49:38 DEBUG GEOparse - SAMPLE: GSM2631244\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631244\n",
            "06-Jan-2023 17:49:38 DEBUG GEOparse - SAMPLE: GSM2631245\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631245\n",
            "06-Jan-2023 17:49:39 DEBUG GEOparse - SAMPLE: GSM2631246\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631246\n",
            "06-Jan-2023 17:49:39 DEBUG GEOparse - SAMPLE: GSM2631247\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631247\n",
            "06-Jan-2023 17:49:39 DEBUG GEOparse - SAMPLE: GSM2631248\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631248\n",
            "06-Jan-2023 17:49:39 DEBUG GEOparse - SAMPLE: GSM2631249\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631249\n",
            "06-Jan-2023 17:49:39 DEBUG GEOparse - SAMPLE: GSM2631250\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631250\n",
            "06-Jan-2023 17:49:39 DEBUG GEOparse - SAMPLE: GSM2631251\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631251\n",
            "06-Jan-2023 17:49:40 DEBUG GEOparse - SAMPLE: GSM2631252\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631252\n",
            "06-Jan-2023 17:49:40 DEBUG GEOparse - SAMPLE: GSM2631253\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631253\n",
            "06-Jan-2023 17:49:40 DEBUG GEOparse - SAMPLE: GSM2631254\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631254\n",
            "06-Jan-2023 17:49:40 DEBUG GEOparse - SAMPLE: GSM2631255\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631255\n",
            "06-Jan-2023 17:49:40 DEBUG GEOparse - SAMPLE: GSM2631256\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631256\n",
            "06-Jan-2023 17:49:40 DEBUG GEOparse - SAMPLE: GSM2631257\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631257\n",
            "06-Jan-2023 17:49:40 DEBUG GEOparse - SAMPLE: GSM2631258\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631258\n",
            "06-Jan-2023 17:49:41 DEBUG GEOparse - SAMPLE: GSM2631259\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631259\n",
            "06-Jan-2023 17:49:41 DEBUG GEOparse - SAMPLE: GSM2631260\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631260\n",
            "06-Jan-2023 17:49:41 DEBUG GEOparse - SAMPLE: GSM2631261\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631261\n",
            "06-Jan-2023 17:49:41 DEBUG GEOparse - SAMPLE: GSM2631262\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631262\n",
            "06-Jan-2023 17:49:41 DEBUG GEOparse - SAMPLE: GSM2631263\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631263\n",
            "06-Jan-2023 17:49:41 DEBUG GEOparse - SAMPLE: GSM2631264\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631264\n",
            "06-Jan-2023 17:49:41 DEBUG GEOparse - SAMPLE: GSM2631265\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631265\n",
            "06-Jan-2023 17:49:42 DEBUG GEOparse - SAMPLE: GSM2631266\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631266\n",
            "06-Jan-2023 17:49:42 DEBUG GEOparse - SAMPLE: GSM2631267\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631267\n",
            "06-Jan-2023 17:49:42 DEBUG GEOparse - SAMPLE: GSM2631268\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631268\n",
            "06-Jan-2023 17:49:42 DEBUG GEOparse - SAMPLE: GSM2631269\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631269\n",
            "06-Jan-2023 17:49:42 DEBUG GEOparse - SAMPLE: GSM2631270\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631270\n",
            "06-Jan-2023 17:49:42 DEBUG GEOparse - SAMPLE: GSM2631271\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631271\n",
            "06-Jan-2023 17:49:43 DEBUG GEOparse - SAMPLE: GSM2631272\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631272\n",
            "06-Jan-2023 17:49:43 DEBUG GEOparse - SAMPLE: GSM2631273\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631273\n",
            "06-Jan-2023 17:49:43 DEBUG GEOparse - SAMPLE: GSM2631274\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631274\n",
            "06-Jan-2023 17:49:43 DEBUG GEOparse - SAMPLE: GSM2631275\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631275\n",
            "06-Jan-2023 17:49:43 DEBUG GEOparse - SAMPLE: GSM2631276\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631276\n",
            "06-Jan-2023 17:49:43 DEBUG GEOparse - SAMPLE: GSM2631277\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631277\n",
            "06-Jan-2023 17:49:44 DEBUG GEOparse - SAMPLE: GSM2631278\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631278\n",
            "06-Jan-2023 17:49:44 DEBUG GEOparse - SAMPLE: GSM2631279\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631279\n",
            "06-Jan-2023 17:49:44 DEBUG GEOparse - SAMPLE: GSM2631280\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631280\n",
            "06-Jan-2023 17:49:44 DEBUG GEOparse - SAMPLE: GSM2631281\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631281\n",
            "06-Jan-2023 17:49:44 DEBUG GEOparse - SAMPLE: GSM2631282\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631282\n",
            "06-Jan-2023 17:49:44 DEBUG GEOparse - SAMPLE: GSM2631283\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631283\n",
            "06-Jan-2023 17:49:45 DEBUG GEOparse - SAMPLE: GSM2631284\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631284\n",
            "06-Jan-2023 17:49:45 DEBUG GEOparse - SAMPLE: GSM2631285\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631285\n",
            "06-Jan-2023 17:49:45 DEBUG GEOparse - SAMPLE: GSM2631286\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631286\n",
            "06-Jan-2023 17:49:45 DEBUG GEOparse - SAMPLE: GSM2631287\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631287\n",
            "06-Jan-2023 17:49:45 DEBUG GEOparse - SAMPLE: GSM2631288\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631288\n",
            "06-Jan-2023 17:49:45 DEBUG GEOparse - SAMPLE: GSM2631289\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631289\n",
            "06-Jan-2023 17:49:45 DEBUG GEOparse - SAMPLE: GSM2631290\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631290\n",
            "06-Jan-2023 17:49:46 DEBUG GEOparse - SAMPLE: GSM2631291\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631291\n",
            "06-Jan-2023 17:49:46 DEBUG GEOparse - SAMPLE: GSM2631292\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631292\n",
            "06-Jan-2023 17:49:46 DEBUG GEOparse - SAMPLE: GSM2631293\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631293\n",
            "06-Jan-2023 17:49:46 DEBUG GEOparse - SAMPLE: GSM2631294\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631294\n",
            "06-Jan-2023 17:49:46 DEBUG GEOparse - SAMPLE: GSM2631295\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631295\n",
            "06-Jan-2023 17:49:46 DEBUG GEOparse - SAMPLE: GSM2631296\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631296\n",
            "06-Jan-2023 17:49:47 DEBUG GEOparse - SAMPLE: GSM2631297\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631297\n",
            "06-Jan-2023 17:49:47 DEBUG GEOparse - SAMPLE: GSM2631298\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631298\n",
            "06-Jan-2023 17:49:47 DEBUG GEOparse - SAMPLE: GSM2631299\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631299\n",
            "06-Jan-2023 17:49:47 DEBUG GEOparse - SAMPLE: GSM2631300\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631300\n",
            "06-Jan-2023 17:49:47 DEBUG GEOparse - SAMPLE: GSM2631301\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631301\n",
            "06-Jan-2023 17:49:47 DEBUG GEOparse - SAMPLE: GSM2631302\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631302\n",
            "06-Jan-2023 17:49:47 DEBUG GEOparse - SAMPLE: GSM2631303\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631303\n",
            "06-Jan-2023 17:49:48 DEBUG GEOparse - SAMPLE: GSM2631304\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631304\n",
            "06-Jan-2023 17:49:48 DEBUG GEOparse - SAMPLE: GSM2631305\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631305\n",
            "06-Jan-2023 17:49:48 DEBUG GEOparse - SAMPLE: GSM2631306\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631306\n",
            "06-Jan-2023 17:49:48 DEBUG GEOparse - SAMPLE: GSM2631307\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631307\n",
            "06-Jan-2023 17:49:48 DEBUG GEOparse - SAMPLE: GSM2631308\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631308\n",
            "06-Jan-2023 17:49:48 DEBUG GEOparse - SAMPLE: GSM2631309\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631309\n",
            "06-Jan-2023 17:49:49 DEBUG GEOparse - SAMPLE: GSM2631310\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631310\n",
            "06-Jan-2023 17:49:49 DEBUG GEOparse - SAMPLE: GSM2631311\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631311\n",
            "06-Jan-2023 17:49:49 DEBUG GEOparse - SAMPLE: GSM2631312\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631312\n",
            "06-Jan-2023 17:49:49 DEBUG GEOparse - SAMPLE: GSM2631313\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631313\n",
            "06-Jan-2023 17:49:49 DEBUG GEOparse - SAMPLE: GSM2631314\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631314\n",
            "06-Jan-2023 17:49:49 DEBUG GEOparse - SAMPLE: GSM2631315\n",
            "DEBUG:GEOparse:SAMPLE: GSM2631315\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[1;30;43mStreaming output truncated to the last 5000 lines.\u001b[0m\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_C07_L3657_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631165/suppl/GSM2631165_Neuro_Krainc-2_P02_C07_L3657_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631166\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_C09_L3666_01\n",
            " - geo_accession : GSM2631166\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3666, disease label: IPD, batch: 03/05/10_L, learning set: VALIDATION, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_C09_L3666_01.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631166/suppl/GSM2631166_Neuro_Krainc-2_P02_C09_L3666_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631167\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_D01_L3671_01\n",
            " - geo_accession : GSM2631167\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3671, disease label: CONTROL, batch: 03/05/10_L, learning set: VALIDATION, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_D01_L3671_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631167/suppl/GSM2631167_Neuro_Krainc-2_P02_D01_L3671_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631168\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_D04_L4028_01\n",
            " - geo_accession : GSM2631168\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L4028, disease label: IPD, batch: 03/05/10_L, learning set: VALIDATION, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_D04_L4028_01.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631168/suppl/GSM2631168_Neuro_Krainc-2_P02_D04_L4028_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631169\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_D07_L3655_01\n",
            " - geo_accession : GSM2631169\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3655, disease label: IPD, batch: 03/05/10_L, learning set: TRAINING, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_D07_L3655_01.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631169/suppl/GSM2631169_Neuro_Krainc-2_P02_D07_L3655_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631170\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_D08_L4029_01\n",
            " - geo_accession : GSM2631170\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L4029, disease label: IPD, batch: 03/05/10_L, learning set: VALIDATION, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_D08_L4029_01.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631170/suppl/GSM2631170_Neuro_Krainc-2_P02_D08_L4029_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631171\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_D10_L2899_01\n",
            " - geo_accession : GSM2631171\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L2899, disease label: ATYPICAL_PD, batch: 03/05/10_L, learning set: VALIDATION, Sex: Male, genetic (mutated pd genes): NONE, age at 1st symptoms: 53, updrs i: 1, updrs ii: 4, updrs iii score on: 19, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 2, hoehn & yahr (off): 0, moca score: 21\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_D10_L2899_01.CEL, Whole blood gene expression from ATYPICAL_PD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631171/suppl/GSM2631171_Neuro_Krainc-2_P02_D10_L2899_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631172\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_D11_L3699_01\n",
            " - geo_accession : GSM2631172\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3699, disease label: CONTROL, batch: 03/05/10_L, learning set: VALIDATION, genetic (mutated pd genes): NMF, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_D11_L3699_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631172/suppl/GSM2631172_Neuro_Krainc-2_P02_D11_L3699_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631173\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_E09_L3058_01\n",
            " - geo_accession : GSM2631173\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3058, disease label: CONTROL, batch: 03/05/10_L, learning set: TRAINING, Sex: Female, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 8, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_E09_L3058_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631173/suppl/GSM2631173_Neuro_Krainc-2_P02_E09_L3058_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631174\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_E12_L4032_01\n",
            " - geo_accession : GSM2631174\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L4032, disease label: PSP, batch: 03/05/10_L, learning set: VALIDATION, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_E12_L4032_01.CEL, Whole blood gene expression from PSP patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631174/suppl/GSM2631174_Neuro_Krainc-2_P02_E12_L4032_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631175\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_F01_L3665_01\n",
            " - geo_accession : GSM2631175\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3665, disease label: CONTROL, batch: 03/05/10_L, learning set: VALIDATION, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_F01_L3665_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631175/suppl/GSM2631175_Neuro_Krainc-2_P02_F01_L3665_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631176\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_F05_L4022_01\n",
            " - geo_accession : GSM2631176\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L4022, disease label: CONTROL, batch: 03/05/10_L, learning set: VALIDATION, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_F05_L4022_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631176/suppl/GSM2631176_Neuro_Krainc-2_P02_F05_L4022_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631177\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_F06_L3656_01\n",
            " - geo_accession : GSM2631177\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3656, disease label: CONTROL, batch: 03/05/10_L, learning set: VALIDATION, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_F06_L3656_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631177/suppl/GSM2631177_Neuro_Krainc-2_P02_F06_L3656_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631178\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_F08_L4031_01\n",
            " - geo_accession : GSM2631178\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L4031, disease label: IPD, batch: 03/05/10_L, learning set: VALIDATION, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_F08_L4031_01.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631178/suppl/GSM2631178_Neuro_Krainc-2_P02_F08_L4031_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631179\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_F09_FRATRO223\n",
            " - geo_accession : GSM2631179\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO223, disease label: CONTROL, batch: 03/05/10_TRO, learning set: TRAINING, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 58, updrs iii score off: 0, hoehn & yahr (off): 0, moca score: 30\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_F09_FRATRO223.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631179/suppl/GSM2631179_Neuro_Krainc-2_P02_F09_FRATRO223.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631180\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_F10_FRATRO306\n",
            " - geo_accession : GSM2631180\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO306, disease label: CONTROL, batch: 03/05/10_TRO, learning set: TRAINING, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 75, updrs iii score off: 1, hoehn & yahr (off): 0, moca score: 26\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_F10_FRATRO306.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631180/suppl/GSM2631180_Neuro_Krainc-2_P02_F10_FRATRO306.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631181\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_F12_FRATRO308_1\n",
            " - geo_accession : GSM2631181\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO308, disease label: CONTROL, batch: 03/05/10_TRO, learning set: TRAINING, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 58, updrs iii score off: 0, hoehn & yahr (off): 0, moca score: 30\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_F12_FRATRO308_1.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631181/suppl/GSM2631181_Neuro_Krainc-2_P02_F12_FRATRO308_1.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631182\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_G01_FRATRO309\n",
            " - geo_accession : GSM2631182\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO309, disease label: IPD, batch: 03/05/10_TRO, learning set: TRAINING, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 70, age at 1st symptoms: 70, updrs i: 3, updrs ii: 8, updrs iii score off: 21, updrs iv: 0, hoehn & yahr (off): 2, moca score: 25\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_G01_FRATRO309.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631182/suppl/GSM2631182_Neuro_Krainc-2_P02_G01_FRATRO309.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631183\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_G02_FRATRO310\n",
            " - geo_accession : GSM2631183\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO310, disease label: IPD, batch: 03/05/10_TRO, learning set: TRAINING, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 59, age at 1st symptoms: 56, updrs i: 4, updrs ii: 10, updrs iii score on: 5, updrs iv: 0, hoehn & yahr (on): 1, moca score: 30\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_G02_FRATRO310.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631183/suppl/GSM2631183_Neuro_Krainc-2_P02_G02_FRATRO310.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631184\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_G03_FRATRO311\n",
            " - geo_accession : GSM2631184\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO311, disease label: IPD, batch: 03/05/10_TRO, learning set: VALIDATION, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 78, age at 1st symptoms: 63, updrs i: 0, updrs ii: 0, updrs iii score on: ND, updrs iii score off: 36, updrs iv: 0, hoehn & yahr (off): 4, moca score: ND\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_G03_FRATRO311.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631184/suppl/GSM2631184_Neuro_Krainc-2_P02_G03_FRATRO311.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631185\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_G04_FRATRO313\n",
            " - geo_accession : GSM2631185\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO313, disease label: CONTROL, batch: 03/05/10_TRO, learning set: TRAINING, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 67, updrs iii score off: 0, hoehn & yahr (off): 0, moca score: 28\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_G04_FRATRO313.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631185/suppl/GSM2631185_Neuro_Krainc-2_P02_G04_FRATRO313.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631186\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_G05_FRATRO314_2\n",
            " - geo_accession : GSM2631186\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO314, disease label: IPD, batch: 03/05/10_TRO, learning set: TRAINING, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 74, age at 1st symptoms: 61, updrs i: 3, updrs ii: 15, updrs iii score on: 31, updrs iv: 6, hoehn & yahr (on): 2.5, moca score: 29\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_G05_FRATRO314_2.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631186/suppl/GSM2631186_Neuro_Krainc-2_P02_G05_FRATRO314_2.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631187\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_G06_FRATRO315\n",
            " - geo_accession : GSM2631187\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO315, disease label: IPD, batch: 03/05/10_TRO, learning set: TRAINING, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 72, age at 1st symptoms: 69, updrs i: 3, updrs ii: 6, updrs iii score on: 43, updrs iv: 1, hoehn & yahr (on): 3, moca score: 24\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_G06_FRATRO315.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631187/suppl/GSM2631187_Neuro_Krainc-2_P02_G06_FRATRO315.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631188\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_G10_FRATRO319\n",
            " - geo_accession : GSM2631188\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO319, disease label: IPD, batch: 03/05/10_TRO, learning set: VALIDATION, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 50, age at 1st symptoms: 40, updrs i: 1, updrs ii: 9, updrs iii score on: 9, updrs iv: 7, hoehn & yahr (on): 2, moca score: 30\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_G10_FRATRO319.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631188/suppl/GSM2631188_Neuro_Krainc-2_P02_G10_FRATRO319.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631189\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_G11_FRATRO320\n",
            " - geo_accession : GSM2631189\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO320, disease label: IPD, batch: 03/05/10_TRO, learning set: TRAINING, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 74, age at 1st symptoms: 72, updrs i: 1, updrs ii: 8, updrs iii score on: 18, updrs iv: 0, hoehn & yahr (on): 2.5, moca score: 29\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_G11_FRATRO320.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631189/suppl/GSM2631189_Neuro_Krainc-2_P02_G11_FRATRO320.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631190\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_H01_FRATRO322\n",
            " - geo_accession : GSM2631190\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO322, disease label: IPD, batch: 03/05/10_TRO, learning set: TRAINING, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 53, age at 1st symptoms: 50, updrs i: 1, updrs ii: 8, updrs iii score on: 24, updrs iv: 1, hoehn & yahr (on): 2, moca score: 28\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_H01_FRATRO322.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631190/suppl/GSM2631190_Neuro_Krainc-2_P02_H01_FRATRO322.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631191\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_A01_FRA_TRO_335\n",
            " - geo_accession : GSM2631191\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO335, disease label: CONTROL, batch: 03/10/10_TRO, learning set: TRAINING, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 57, updrs iii score off: 1, hoehn & yahr (off): 0, moca score: 28\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_A01_FRA_TRO_335.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631191/suppl/GSM2631191_Neuro_Krainc-2_P03_A01_FRA_TRO_335.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631192\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_A04_FRA_TRO_338_01\n",
            " - geo_accession : GSM2631192\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO338, disease label: IPD, batch: 03/10/10_TRO, learning set: TRAINING, Sex: Female, age at exam: 60, age at 1st symptoms: 55, updrs i: 0, updrs ii: 5, updrs iii score on: ND, updrs iii score off: 32, updrs iv: 2, hoehn & yahr (on): ND, hoehn & yahr (off): 0, moca score: ND\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_A04_FRA_TRO_338_01.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631192/suppl/GSM2631192_Neuro_Krainc-2_P03_A04_FRA_TRO_338_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631193\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_A05_FRA_TRO_339_01\n",
            " - geo_accession : GSM2631193\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO339, disease label: GPD, batch: 03/10/10_TRO, learning set: VALIDATION, Sex: Female, genetic (mutated pd genes): LRRK2, age at exam: 42, age at 1st symptoms: 31, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_A05_FRA_TRO_339_01.CEL, Whole blood gene expression from GPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631193/suppl/GSM2631193_Neuro_Krainc-2_P03_A05_FRA_TRO_339_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631194\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_A06_FRA_TRO_340_01\n",
            " - geo_accession : GSM2631194\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO340, disease label: CONTROL, batch: 03/10/10_TRO, learning set: TRAINING, Sex: Male, age at exam: 42, age at 1st symptoms: NA, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_A06_FRA_TRO_340_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631194/suppl/GSM2631194_Neuro_Krainc-2_P03_A06_FRA_TRO_340_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631195\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_A07_FRA_TRO_341_01\n",
            " - geo_accession : GSM2631195\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO341, disease label: CONTROL, batch: 03/10/10_TRO, learning set: TRAINING, Sex: Male, age at exam: 49, age at 1st symptoms: NA, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_A07_FRA_TRO_341_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631195/suppl/GSM2631195_Neuro_Krainc-2_P03_A07_FRA_TRO_341_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631196\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_A08_FRA_TRO_342_01\n",
            " - geo_accession : GSM2631196\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO342, disease label: CONTROL, batch: 03/10/10_TRO, learning set: TRAINING, Sex: Male, age at exam: 40, age at 1st symptoms: NA, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_A08_FRA_TRO_342_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631196/suppl/GSM2631196_Neuro_Krainc-2_P03_A08_FRA_TRO_342_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631197\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_A09_FRA_TRO_343_01\n",
            " - geo_accession : GSM2631197\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO343, disease label: CONTROL, batch: 03/10/10_TRO, learning set: TRAINING, Sex: Male, age at exam: 43, age at 1st symptoms: NA, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_A09_FRA_TRO_343_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631197/suppl/GSM2631197_Neuro_Krainc-2_P03_A09_FRA_TRO_343_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631198\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_A10_FRA_TRO_344_01\n",
            " - geo_accession : GSM2631198\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO344, disease label: CONTROL, batch: 03/10/10_TRO, learning set: TRAINING, Sex: Male, age at exam: 40, age at 1st symptoms: NA, updrs iii score off: 0, hoehn & yahr (off): 0, moca score: 30\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_A10_FRA_TRO_344_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631198/suppl/GSM2631198_Neuro_Krainc-2_P03_A10_FRA_TRO_344_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631199\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_A11_FRA_TRO_345_01\n",
            " - geo_accession : GSM2631199\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO345, disease label: MSA, batch: 03/10/10_TRO, learning set: VALIDATION, Sex: Female, age at exam: 81, age at 1st symptoms: 74, updrs i: 1, updrs ii: 26, updrs iii score off: 55, updrs iv: 1, hoehn & yahr (off): 5, moca score: 27\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_A11_FRA_TRO_345_01.CEL, Whole blood gene expression from MSA patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631199/suppl/GSM2631199_Neuro_Krainc-2_P03_A11_FRA_TRO_345_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631200\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_A12_FRA_TRO_346_01\n",
            " - geo_accession : GSM2631200\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO346, disease label: IPD, batch: 03/10/10_TRO, learning set: TRAINING, Sex: Female, age at exam: 51, age at 1st symptoms: 47, updrs iii score off: 18, hoehn & yahr (off): 1\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_A12_FRA_TRO_346_01.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631200/suppl/GSM2631200_Neuro_Krainc-2_P03_A12_FRA_TRO_346_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631201\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_B01_FRA_TRO_347_01\n",
            " - geo_accession : GSM2631201\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO347, disease label: GPD, batch: 03/10/10_TRO, learning set: VALIDATION, Sex: Female, genetic (mutated pd genes): PARKIN, age at exam: 58, age at 1st symptoms: 30, updrs i: 0, updrs ii: 2, updrs iii score on: 8, updrs iv: 0, hoehn & yahr (on): 3, moca score: 28\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_B01_FRA_TRO_347_01.CEL, Whole blood gene expression from GPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631201/suppl/GSM2631201_Neuro_Krainc-2_P03_B01_FRA_TRO_347_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631202\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_B03_FRA_TRO_349_01\n",
            " - geo_accession : GSM2631202\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO349, disease label: GPD, batch: 03/11/10_TRO, learning set: VALIDATION, Sex: Female, genetic (mutated pd genes): PARKIN, age at exam: 42, age at 1st symptoms: 20, updrs i: 3, updrs ii: 10, updrs iii score off: 28, updrs iv: 12, hoehn & yahr (off): 3, moca score: 27\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_B03_FRA_TRO_349_01.CEL, Whole blood gene expression from GPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631202/suppl/GSM2631202_Neuro_Krainc-2_P03_B03_FRA_TRO_349_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631203\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_B05_FRA_TRO_351_01\n",
            " - geo_accession : GSM2631203\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO351, disease label: GPD, batch: 03/10/10_TRO, learning set: VALIDATION, Sex: Female, genetic (mutated pd genes): PARKIN, age at exam: 50, age at 1st symptoms: 36, updrs i: 5, updrs ii: 1, updrs iii score on: 4, updrs iv: 4, hoehn & yahr (on): 1, moca score: 24\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_B05_FRA_TRO_351_01.CEL, Whole blood gene expression from GPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631203/suppl/GSM2631203_Neuro_Krainc-2_P03_B05_FRA_TRO_351_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631204\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_B09_FRA_TRO_355_01\n",
            " - geo_accession : GSM2631204\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO355, disease label: MSA, batch: 03/10/10_TRO, learning set: VALIDATION, Sex: Female, age at exam: 75, age at 1st symptoms: 71, updrs i: 5, updrs ii: 28, updrs iii score off: 64, updrs iv: 1, hoehn & yahr (off): 5\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_B09_FRA_TRO_355_01.CEL, Whole blood gene expression from MSA patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631204/suppl/GSM2631204_Neuro_Krainc-2_P03_B09_FRA_TRO_355_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631205\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_B12_FRA_TRO_359_01\n",
            " - geo_accession : GSM2631205\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO359, disease label: GPD, batch: 03/10/10_TRO, learning set: VALIDATION, Sex: Male, genetic (mutated pd genes): PARKIN, age at exam: 62, age at 1st symptoms: 28, updrs i: 2, updrs ii: 10, updrs iii score on: 38, updrs iv: 10, hoehn & yahr (on): 3, moca score: 28\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_B12_FRA_TRO_359_01.CEL, Whole blood gene expression from GPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631205/suppl/GSM2631205_Neuro_Krainc-2_P03_B12_FRA_TRO_359_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631206\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_C02_FRA_TRO_361_01\n",
            " - geo_accession : GSM2631206\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO361, disease label: GPD, batch: 03/10/10_TRO, learning set: VALIDATION, Sex: Male, genetic (mutated pd genes): PARKIN, age at exam: 48, age at 1st symptoms: 26, updrs iii score on: 1, hoehn & yahr (on): 1\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_C02_FRA_TRO_361_01.CEL, Whole blood gene expression from GPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631206/suppl/GSM2631206_Neuro_Krainc-2_P03_C02_FRA_TRO_361_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631207\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_C04_FRA_TRO_363_01\n",
            " - geo_accession : GSM2631207\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO363, disease label: GPD, batch: 03/10/10_TRO, learning set: VALIDATION, Sex: Female, genetic (mutated pd genes): PARKIN, age at exam: 66, age at 1st symptoms: 30, updrs i: 2, updrs ii: 14, updrs iii score on: 37, updrs iv: 6, hoehn & yahr (on): 3, moca score: 30\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_C04_FRA_TRO_363_01.CEL, Whole blood gene expression from GPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631207/suppl/GSM2631207_Neuro_Krainc-2_P03_C04_FRA_TRO_363_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631208\n",
            "Metadata:\n",
            " - title : E7R_037b04\n",
            " - geo_accession : GSM2631208\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, disease label: HD_HD_BATCH, batch: 01/09/08_L, learning set: TEST, genetic (mutated pd genes): NA, age at exam: NA, age at 1st symptoms: NA, updrs i: NA, updrs ii: NA, updrs iii score on: NA, updrs iii score off: NA, updrs iv: NA, hoehn & yahr (on): NA, hoehn & yahr (off): NA, moca score: NA\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E7R_037b04_(HG-U133_Plus_2).CEL, Whole blood gene expression from HD_HD_BATCH patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631208/suppl/GSM2631208_E7R_037b04_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631209\n",
            "Metadata:\n",
            " - title : E7R_037b05\n",
            " - geo_accession : GSM2631209\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, disease label: HD_HD_BATCH, batch: 01/09/08_L, learning set: TEST, genetic (mutated pd genes): NA, age at exam: NA, age at 1st symptoms: NA, updrs i: NA, updrs ii: NA, updrs iii score on: NA, updrs iii score off: NA, updrs iv: NA, hoehn & yahr (on): NA, hoehn & yahr (off): NA, moca score: NA\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E7R_037b05_(HG-U133_Plus_2).CEL, Whole blood gene expression from HD_HD_BATCH patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631209/suppl/GSM2631209_E7R_037b05_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631210\n",
            "Metadata:\n",
            " - title : E7R_037b06\n",
            " - geo_accession : GSM2631210\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, disease label: HD_HD_BATCH, batch: 01/15/08_L, learning set: TEST, genetic (mutated pd genes): NA, age at exam: NA, age at 1st symptoms: NA, updrs i: NA, updrs ii: NA, updrs iii score on: NA, updrs iii score off: NA, updrs iv: NA, hoehn & yahr (on): NA, hoehn & yahr (off): NA, moca score: NA\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E7R_037b06_(HG-U133_Plus_2).CEL, Whole blood gene expression from HD_HD_BATCH patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631210/suppl/GSM2631210_E7R_037b06_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631211\n",
            "Metadata:\n",
            " - title : E7R_037c16\n",
            " - geo_accession : GSM2631211\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, disease label: HD_HD_BATCH, batch: 01/17/08_L, learning set: TEST, genetic (mutated pd genes): NA, age at exam: NA, age at 1st symptoms: NA, updrs i: NA, updrs ii: NA, updrs iii score on: NA, updrs iii score off: NA, updrs iv: NA, hoehn & yahr (on): NA, hoehn & yahr (off): NA, moca score: NA\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E7R_037c16_(HG-U133_Plus_2).CEL, Whole blood gene expression from HD_HD_BATCH patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631211/suppl/GSM2631211_E7R_037c16_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631212\n",
            "Metadata:\n",
            " - title : E7R_037c17\n",
            " - geo_accession : GSM2631212\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, disease label: HD_HD_BATCH, batch: 01/18/08_L, learning set: TEST, genetic (mutated pd genes): NA, age at exam: NA, age at 1st symptoms: NA, updrs i: NA, updrs ii: NA, updrs iii score on: NA, updrs iii score off: NA, updrs iv: NA, hoehn & yahr (on): NA, hoehn & yahr (off): NA, moca score: NA\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E7R_037c17_(HG-U133_Plus_2).CEL, Whole blood gene expression from HD_HD_BATCH patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631212/suppl/GSM2631212_E7R_037c17_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631213\n",
            "Metadata:\n",
            " - title : E7R_037c23\n",
            " - geo_accession : GSM2631213\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, disease label: HD_HD_BATCH, batch: 01/17/08_L, learning set: TEST, genetic (mutated pd genes): NA, age at exam: NA, age at 1st symptoms: NA, updrs i: NA, updrs ii: NA, updrs iii score on: NA, updrs iii score off: NA, updrs iv: NA, hoehn & yahr (on): NA, hoehn & yahr (off): NA, moca score: NA\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E7R_037c23_(HG-U133_Plus_2).CEL, Whole blood gene expression from HD_HD_BATCH patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631213/suppl/GSM2631213_E7R_037c23_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631214\n",
            "Metadata:\n",
            " - title : E7R_037d06\n",
            " - geo_accession : GSM2631214\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, disease label: HD_HD_BATCH, batch: 01/16/08_L, learning set: TEST, genetic (mutated pd genes): NA, age at exam: NA, age at 1st symptoms: NA, updrs i: NA, updrs ii: NA, updrs iii score on: NA, updrs iii score off: NA, updrs iv: NA, hoehn & yahr (on): NA, hoehn & yahr (off): NA, moca score: NA\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E7R_037d06_(HG-U133_Plus_2).CEL, Whole blood gene expression from HD_HD_BATCH patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631214/suppl/GSM2631214_E7R_037d06_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631215\n",
            "Metadata:\n",
            " - title : E7R_037d12\n",
            " - geo_accession : GSM2631215\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, disease label: HD_HD_BATCH, batch: 01/18/08_L, learning set: TEST, genetic (mutated pd genes): NA, age at exam: NA, age at 1st symptoms: NA, updrs i: NA, updrs ii: NA, updrs iii score on: NA, updrs iii score off: NA, updrs iv: NA, hoehn & yahr (on): NA, hoehn & yahr (off): NA, moca score: NA\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E7R_037d12_(HG-U133_Plus_2).CEL, Whole blood gene expression from HD_HD_BATCH patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631215/suppl/GSM2631215_E7R_037d12_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631216\n",
            "Metadata:\n",
            " - title : E7R_039a100\n",
            " - geo_accession : GSM2631216\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L2585, disease label: PSP, batch: 02/22/08_L, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at 1st symptoms: 72, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 9, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E7R_039a100_(HG-U133_Plus_2).CEL, Whole blood gene expression from PSP patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631216/suppl/GSM2631216_E7R_039a100_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631217\n",
            "Metadata:\n",
            " - title : E7R_039a43\n",
            " - geo_accession : GSM2631217\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L2130, disease label: GENETIC_UNAFFECTED, batch: 02/04/08_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): PINK1, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 29\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E7R_039a43_(HG-U133_Plus_2).CEL, Whole blood gene expression from GENETIC_UNAFFECTED patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631217/suppl/GSM2631217_E7R_039a43_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631218\n",
            "Metadata:\n",
            " - title : E7R_039a47\n",
            " - geo_accession : GSM2631218\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L2128, disease label: GPD, batch: 02/07/08_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): PINK1, updrs i: 0, updrs ii: 0, updrs iii score on: 1, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 28\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E7R_039a47_(HG-U133_Plus_2).CEL, Whole blood gene expression from GPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631218/suppl/GSM2631218_E7R_039a47_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631219\n",
            "Metadata:\n",
            " - title : E7R_039a53\n",
            " - geo_accession : GSM2631219\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L2131, disease label: ATYPICAL_PD, batch: 02/07/08_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E7R_039a53_(HG-U133_Plus_2).CEL, Whole blood gene expression from ATYPICAL_PD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631219/suppl/GSM2631219_E7R_039a53_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631220\n",
            "Metadata:\n",
            " - title : E7R_039a61\n",
            " - geo_accession : GSM2631220\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L2837, disease label: IPD, batch: 02/08/08_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at 1st symptoms: 64, updrs i: 0, updrs ii: 0, updrs iii score on: 42, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 9, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E7R_039a61_(HG-U133_Plus_2).CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631220/suppl/GSM2631220_E7R_039a61_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631221\n",
            "Metadata:\n",
            " - title : E7R_039a66\n",
            " - geo_accession : GSM2631221\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L1591, disease label: CONTROL, batch: 02/08/08_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E7R_039a66_(HG-U133_Plus_2).CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631221/suppl/GSM2631221_E7R_039a66_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631222\n",
            "Metadata:\n",
            " - title : E7R_039a86\n",
            " - geo_accession : GSM2631222\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3056, disease label: GENETIC_UNAFFECTED, batch: 02/12/08_L, learning set: TEST, Sex: Female, genetic (mutated pd genes): PARKIN, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E7R_039a86_(HG-U133_Plus_2).CEL, Whole blood gene expression from GENETIC_UNAFFECTED patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631222/suppl/GSM2631222_E7R_039a86_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631223\n",
            "Metadata:\n",
            " - title : E8R_012a10\n",
            " - geo_accession : GSM2631223\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3078, disease label: PD_DEMENTIA, batch: 05/16/08_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at 1st symptoms: 69, updrs i: 1, updrs ii: 7, updrs iii score on: 26, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 3, hoehn & yahr (off): 0, moca score: 20\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E8R_012a10_(HG-U133_Plus_2).CEL, Whole blood gene expression from PD_DEMENTIA patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631223/suppl/GSM2631223_E8R_012a10_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631224\n",
            "Metadata:\n",
            " - title : E8R_012a15\n",
            " - geo_accession : GSM2631224\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3208, disease label: IPD, batch: 05/27/08_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E8R_012a15_(HG-U133_Plus_2).CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631224/suppl/GSM2631224_E8R_012a15_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631225\n",
            "Metadata:\n",
            " - title : E8R_012a28_2\n",
            " - geo_accession : GSM2631225\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3258, disease label: IPD, batch: 06/19/08_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at 1st symptoms: 54, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 9, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : E8R_012a28_2_(HG-U133_Plus_2).CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631225/suppl/GSM2631225_E8R_012a28_2_HG-U133_Plus_2_.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631226\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P01_B9_L3292\n",
            " - geo_accession : GSM2631226\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3292, disease label: GPD, batch: 08/28/09_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): ATP13A2, age at 1st symptoms: 10, updrs i: 6, updrs ii: 35, updrs iii score on: 51, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 5, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P01_B9_L3292.CEL, Whole blood gene expression from GPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631226/suppl/GSM2631226_Neuro_Krainc_P01_B9_L3292.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631227\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P01_C8_L3338\n",
            " - geo_accession : GSM2631227\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3338, disease label: CONTROL, batch: 08/28/09_L, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P01_C8_L3338.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631227/suppl/GSM2631227_Neuro_Krainc_P01_C8_L3338.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631228\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P01_D4_L3348\n",
            " - geo_accession : GSM2631228\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3348, disease label: GPD, batch: 08/28/09_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): SNCA, updrs i: 1, updrs ii: 3, updrs iii score on: 3, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 8, hoehn & yahr (off): 0, moca score: 29\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P01_D4_L3348.CEL, Whole blood gene expression from GPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631228/suppl/GSM2631228_Neuro_Krainc_P01_D4_L3348.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631229\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P01_E4_L3365\n",
            " - geo_accession : GSM2631229\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3365, disease label: GENETIC_UNAFFECTED, batch: 08/28/09_L, learning set: TEST, Sex: Female, genetic (mutated pd genes): PINK1, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 8, hoehn & yahr (off): 0, moca score: 30\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P01_E4_L3365.CEL, Whole blood gene expression from GENETIC_UNAFFECTED patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631229/suppl/GSM2631229_Neuro_Krainc_P01_E4_L3365.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631230\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P01_G1_L3453\n",
            " - geo_accession : GSM2631230\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3453, disease label: CONTROL, batch: 08/28/09_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P01_G1_L3453.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631230/suppl/GSM2631230_Neuro_Krainc_P01_G1_L3453.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631231\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P01_G11_L3463\n",
            " - geo_accession : GSM2631231\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3463, disease label: CONTROL, batch: 08/28/09_L, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P01_G11_L3463.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631231/suppl/GSM2631231_Neuro_Krainc_P01_G11_L3463.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631232\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P01_H8_L3493\n",
            " - geo_accession : GSM2631232\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3493, disease label: CONTROL, batch: 08/28/09_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P01_H8_L3493.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631232/suppl/GSM2631232_Neuro_Krainc_P01_H8_L3493.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631233\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_A04_L3506\n",
            " - geo_accession : GSM2631233\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3506, disease label: PD_DEMENTIA, batch: 09/01/09_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 2, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 8, hoehn & yahr (off): 0, moca score: 29\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_A04_L3506.CEL, Whole blood gene expression from PD_DEMENTIA patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631233/suppl/GSM2631233_Neuro_Krainc_P02_A04_L3506.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631234\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_A08_L3510\n",
            " - geo_accession : GSM2631234\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3510, disease label: CONTROL, batch: 09/01/09_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_A08_L3510.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631234/suppl/GSM2631234_Neuro_Krainc_P02_A08_L3510.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631235\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_A11_L3514\n",
            " - geo_accession : GSM2631235\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3514, disease label: CONTROL, batch: 09/01/09_L, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_A11_L3514.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631235/suppl/GSM2631235_Neuro_Krainc_P02_A11_L3514.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631236\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_A12_L3515\n",
            " - geo_accession : GSM2631236\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3515, disease label: CONTROL, batch: 09/01/09_L, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_A12_L3515.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631236/suppl/GSM2631236_Neuro_Krainc_P02_A12_L3515.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631237\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_B01_L3516\n",
            " - geo_accession : GSM2631237\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3516, disease label: CONTROL, batch: 09/01/09_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_B01_L3516.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631237/suppl/GSM2631237_Neuro_Krainc_P02_B01_L3516.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631238\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_B04_L3519\n",
            " - geo_accession : GSM2631238\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3519, disease label: CONTROL, batch: 09/01/09_L, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_B04_L3519.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631238/suppl/GSM2631238_Neuro_Krainc_P02_B04_L3519.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631239\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_B06_L3521_2\n",
            " - geo_accession : GSM2631239\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3521, disease label: CONTROL, batch: 09/01/09_L, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_B06_L3521_2.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631239/suppl/GSM2631239_Neuro_Krainc_P02_B06_L3521_2.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631240\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_B07_L3522\n",
            " - geo_accession : GSM2631240\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3522, disease label: CONTROL, batch: 09/01/09_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_B07_L3522.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631240/suppl/GSM2631240_Neuro_Krainc_P02_B07_L3522.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631241\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_B12_L3603\n",
            " - geo_accession : GSM2631241\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3603, disease label: CONTROL, batch: 09/01/09_L, learning set: TEST, Sex: Female, genetic (mutated pd genes): NMF, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_B12_L3603.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631241/suppl/GSM2631241_Neuro_Krainc_P02_B12_L3603.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631242\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_C03_L3651\n",
            " - geo_accession : GSM2631242\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3651, disease label: CONTROL, batch: 09/01/09_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_C03_L3651.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631242/suppl/GSM2631242_Neuro_Krainc_P02_C03_L3651.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631243\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_C04_L3652\n",
            " - geo_accession : GSM2631243\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3652, disease label: CONTROL, batch: 09/01/09_L, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_C04_L3652.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631243/suppl/GSM2631243_Neuro_Krainc_P02_C04_L3652.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631244\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_C05_L3712\n",
            " - geo_accession : GSM2631244\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3712, disease label: IPD, batch: 09/02/09_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_C05_L3712.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631244/suppl/GSM2631244_Neuro_Krainc_P02_C05_L3712.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631245\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_C06_L3713\n",
            " - geo_accession : GSM2631245\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3713, disease label: IPD, batch: 09/01/09_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_C06_L3713.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631245/suppl/GSM2631245_Neuro_Krainc_P02_C06_L3713.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631246\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_C07_L3714\n",
            " - geo_accession : GSM2631246\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3714, disease label: IPD, batch: 09/01/09_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_C07_L3714.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631246/suppl/GSM2631246_Neuro_Krainc_P02_C07_L3714.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631247\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_C09_L3716_2\n",
            " - geo_accession : GSM2631247\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3716, disease label: IPD, batch: 09/01/09_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_C09_L3716_2.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631247/suppl/GSM2631247_Neuro_Krainc_P02_C09_L3716_2.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631248\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_C10_L3717\n",
            " - geo_accession : GSM2631248\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3717, disease label: CONTROL, batch: 09/01/09_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_C10_L3717.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631248/suppl/GSM2631248_Neuro_Krainc_P02_C10_L3717.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631249\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_C12_L3731\n",
            " - geo_accession : GSM2631249\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3731, disease label: IPD, batch: 09/01/09_L, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_C12_L3731.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631249/suppl/GSM2631249_Neuro_Krainc_P02_C12_L3731.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631250\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_D01_B25_RPT\n",
            " - geo_accession : GSM2631250\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Feb 10 2019\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: B25, disease label: GENETIC_UNAFFECTED, batch: 09/16/09_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): PARKIN, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_D01_B25_RPT.CEL, Whole blood gene expression from GENETIC_UNAFFECTED patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631250/suppl/GSM2631250_Neuro_Krainc_P02_D01_B25_RPT.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631251\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_D08_B45\n",
            " - geo_accession : GSM2631251\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: B45, disease label: GPD, batch: 09/01/09_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): PARKIN, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_D08_B45.CEL, Whole blood gene expression from GPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631251/suppl/GSM2631251_Neuro_Krainc_P02_D08_B45.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631252\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_D09_B980\n",
            " - geo_accession : GSM2631252\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: B980, disease label: CONTROL, batch: 09/01/09_L, learning set: TEST, Sex: Female, genetic (mutated pd genes): NMF, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_D09_B980.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631252/suppl/GSM2631252_Neuro_Krainc_P02_D09_B980.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631253\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_E01_FRATRO5\n",
            " - geo_accession : GSM2631253\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO5, disease label: IPD, batch: 09/01/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 58, age at 1st symptoms: 52, updrs i: 1, updrs ii: 13, updrs iii score on: 17, updrs iv: 0, hoehn & yahr (on): 2, moca score: 30\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_E01_FRATRO5.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631253/suppl/GSM2631253_Neuro_Krainc_P02_E01_FRATRO5.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631254\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_E02_FRATRO6_2\n",
            " - geo_accession : GSM2631254\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO6, disease label: IPD, batch: 09/01/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 54, age at 1st symptoms: 45, updrs i: 0, updrs ii: 7, updrs iii score on: 25, updrs iv: 0, hoehn & yahr (on): 2, moca score: 24\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_E02_FRATRO6_2.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631254/suppl/GSM2631254_Neuro_Krainc_P02_E02_FRATRO6_2.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631255\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_E03_FRATRO7\n",
            " - geo_accession : GSM2631255\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO7, disease label: GPD, batch: 09/01/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): LRRK2, age at exam: 49, age at 1st symptoms: 39, updrs i: 3, updrs ii: 26, updrs iii score on: 19, updrs iv: 9, hoehn & yahr (on): 2, hoehn & yahr (off): 5, moca score: 27\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_E03_FRATRO7.CEL, Whole blood gene expression from GPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631255/suppl/GSM2631255_Neuro_Krainc_P02_E03_FRATRO7.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631256\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_E11_FRATRO16\n",
            " - geo_accession : GSM2631256\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO16, disease label: IPD, batch: 09/01/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 63, age at 1st symptoms: 51, updrs i: 1, updrs ii: 17, updrs iii score off: 33, updrs iv: 13, hoehn & yahr (on): 3, hoehn & yahr (off): 4, moca score: 28\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_E11_FRATRO16.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631256/suppl/GSM2631256_Neuro_Krainc_P02_E11_FRATRO16.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631257\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_F03_FRATRO21\n",
            " - geo_accession : GSM2631257\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO21, disease label: IPD, batch: 09/01/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 57, age at 1st symptoms: 49, updrs i: 4, updrs ii: 11, updrs iii score off: 23, updrs iv: 4, hoehn & yahr (off): 2.5, moca score: 30\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_F03_FRATRO21.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631257/suppl/GSM2631257_Neuro_Krainc_P02_F03_FRATRO21.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631258\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_F06_FRATRO24\n",
            " - geo_accession : GSM2631258\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO24, disease label: IPD, batch: 09/01/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 42, age at 1st symptoms: 36, updrs i: 1, updrs ii: 5, updrs iii score on: 6, updrs iv: 1, hoehn & yahr (on): 1, moca score: 29\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_F06_FRATRO24.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631258/suppl/GSM2631258_Neuro_Krainc_P02_F06_FRATRO24.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631259\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_F07_FRATRO26\n",
            " - geo_accession : GSM2631259\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO26, disease label: IPD, batch: 09/01/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 67, age at 1st symptoms: 65, updrs i: 0, updrs ii: 4, updrs iii score on: 14, updrs iv: 0, hoehn & yahr (on): 1, moca score: ND\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_F07_FRATRO26.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631259/suppl/GSM2631259_Neuro_Krainc_P02_F07_FRATRO26.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631260\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_F08_FRATRO28\n",
            " - geo_accession : GSM2631260\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO28, disease label: IPD, batch: 09/01/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 54, age at 1st symptoms: 47, updrs i: 1, updrs ii: 7, updrs iii score on: 16, updrs iv: 1, hoehn & yahr (on): 1, moca score: 30\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_F08_FRATRO28.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631260/suppl/GSM2631260_Neuro_Krainc_P02_F08_FRATRO28.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631261\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_F09_FRATRO29\n",
            " - geo_accession : GSM2631261\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO29, disease label: IPD, batch: 09/01/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 74, age at 1st symptoms: 69, updrs i: 4, updrs ii: 3, updrs iii score on: 20, updrs iv: 4, hoehn & yahr (on): 2.5, moca score: 29\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_F09_FRATRO29.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631261/suppl/GSM2631261_Neuro_Krainc_P02_F09_FRATRO29.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631262\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_F10_FRATRO30\n",
            " - geo_accession : GSM2631262\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO30, disease label: IPD, batch: 09/01/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 66, age at 1st symptoms: 62, updrs i: 4, updrs ii: 17, updrs iii score off: 46, updrs iv: 4, hoehn & yahr (off): 2.5, moca score: 25\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_F10_FRATRO30.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631262/suppl/GSM2631262_Neuro_Krainc_P02_F10_FRATRO30.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631263\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_F12_FRATRO32\n",
            " - geo_accession : GSM2631263\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO32, disease label: IPD, batch: 09/01/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 47, age at 1st symptoms: 44, updrs i: 0, updrs ii: 2, updrs iii score on: 10, updrs iv: 2, hoehn & yahr (on): 1, moca score: 28\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_F12_FRATRO32.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631263/suppl/GSM2631263_Neuro_Krainc_P02_F12_FRATRO32.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631264\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_G02_FRATRO34\n",
            " - geo_accession : GSM2631264\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO34, disease label: IPD, batch: 09/01/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 71, age at 1st symptoms: 67, updrs i: 3, updrs ii: 6, updrs iii score off: 33, updrs iv: 2, hoehn & yahr (off): 1.5, moca score: 19\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_G02_FRATRO34.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631264/suppl/GSM2631264_Neuro_Krainc_P02_G02_FRATRO34.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631265\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_G03_FRATRO35\n",
            " - geo_accession : GSM2631265\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO35, disease label: IPD, batch: 09/01/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 70, age at 1st symptoms: 64, updrs i: 1, updrs ii: 8, updrs iii score off: 17, updrs iv: 4, hoehn & yahr (off): 1.5, moca score: 28\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_G03_FRATRO35.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631265/suppl/GSM2631265_Neuro_Krainc_P02_G03_FRATRO35.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631266\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_G04_FRATRO36\n",
            " - geo_accession : GSM2631266\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO36, disease label: IPD, batch: 09/01/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 67, age at 1st symptoms: 63, updrs i: 4, updrs ii: 11, updrs iii score on: 11, updrs iv: 3, hoehn & yahr (on): 1, moca score: 27\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_G04_FRATRO36.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631266/suppl/GSM2631266_Neuro_Krainc_P02_G04_FRATRO36.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631267\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_G05_FRATRO37\n",
            " - geo_accession : GSM2631267\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO37, disease label: IPD, batch: 09/01/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 69, age at 1st symptoms: 64, updrs i: 1, updrs ii: 19, updrs iii score on: 29, updrs iv: 0, hoehn & yahr (on): 2, moca score: 28\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_G05_FRATRO37.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631267/suppl/GSM2631267_Neuro_Krainc_P02_G05_FRATRO37.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631268\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_G09_FRATRO41\n",
            " - geo_accession : GSM2631268\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO41, disease label: IPD, batch: 09/02/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 77, age at 1st symptoms: 64, updrs i: 0, updrs ii: 13, updrs iii score on: 26, updrs iv: 1, hoehn & yahr (on): 2, moca score: 29\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_G09_FRATRO41.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631268/suppl/GSM2631268_Neuro_Krainc_P02_G09_FRATRO41.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631269\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P02_H11_FRATRO55\n",
            " - geo_accession : GSM2631269\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO55, disease label: IPD, batch: 09/02/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 70, age at 1st symptoms: 67, updrs i: 2, updrs ii: 4, updrs iii score on: 29, updrs iv: 1, hoehn & yahr (on): ND, moca score: 27\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P02_H11_FRATRO55.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631269/suppl/GSM2631269_Neuro_Krainc_P02_H11_FRATRO55.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631270\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P03_A06_FRATRO62\n",
            " - geo_accession : GSM2631270\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO62, disease label: HD, batch: 08/20/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 54, age at 1st symptoms: 50, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P03_A06_FRATRO62.CEL, Whole blood gene expression from HD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631270/suppl/GSM2631270_Neuro_Krainc_P03_A06_FRATRO62.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631271\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P03_B04_FRATRO077\n",
            " - geo_accession : GSM2631271\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO77, disease label: IPD, batch: 08/25/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 62, age at 1st symptoms: 57, updrs i: 0, updrs ii: 6, updrs iii score on: 21, updrs iv: 3, hoehn & yahr (on): 2, moca score: 30\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P03_B04_FRATRO077.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631271/suppl/GSM2631271_Neuro_Krainc_P03_B04_FRATRO077.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631272\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P03_B06_FRATRO079\n",
            " - geo_accession : GSM2631272\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO79, disease label: CONTROL, batch: 08/25/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 74, updrs iii score off: 0, hoehn & yahr (off): 0, moca score: 29\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P03_B06_FRATRO079.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631272/suppl/GSM2631272_Neuro_Krainc_P03_B06_FRATRO079.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631273\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P03_C03_FRATRO093\n",
            " - geo_accession : GSM2631273\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO93, disease label: CONTROL, batch: 08/25/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 52, updrs iii score off: 0, hoehn & yahr (off): 0, moca score: 25\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P03_C03_FRATRO093.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631273/suppl/GSM2631273_Neuro_Krainc_P03_C03_FRATRO093.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631274\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P03_E07_FRATRO121\n",
            " - geo_accession : GSM2631274\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO121, disease label: CONTROL, batch: 08/26/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 78, updrs iii score off: 1, hoehn & yahr (off): 0, moca score: 28\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P03_E07_FRATRO121.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631274/suppl/GSM2631274_Neuro_Krainc_P03_E07_FRATRO121.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631275\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P03_E08_FRATRO122\n",
            " - geo_accession : GSM2631275\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO122, disease label: CONTROL, batch: 08/26/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 74, updrs iii score off: 0, hoehn & yahr (off): 0, moca score: 24\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P03_E08_FRATRO122.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631275/suppl/GSM2631275_Neuro_Krainc_P03_E08_FRATRO122.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631276\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P03_E09_FRATRO123\n",
            " - geo_accession : GSM2631276\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO123, disease label: CONTROL, batch: 08/26/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 70, updrs iii score off: 2, hoehn & yahr (off): 0, moca score: 26\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P03_E09_FRATRO123.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631276/suppl/GSM2631276_Neuro_Krainc_P03_E09_FRATRO123.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631277\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P03_E10_FRATRO124\n",
            " - geo_accession : GSM2631277\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO124, disease label: CONTROL, batch: 08/26/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 69, updrs iii score off: 0, hoehn & yahr (off): 0, moca score: 28\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P03_E10_FRATRO124.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631277/suppl/GSM2631277_Neuro_Krainc_P03_E10_FRATRO124.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631278\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P03_F01_FRATRO127\n",
            " - geo_accession : GSM2631278\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO127, disease label: CONTROL, batch: 08/26/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 75, updrs iii score off: 0, hoehn & yahr (off): 0, moca score: 29\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P03_F01_FRATRO127.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631278/suppl/GSM2631278_Neuro_Krainc_P03_F01_FRATRO127.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631279\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P03_F02_FRATRO129\n",
            " - geo_accession : GSM2631279\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO129, disease label: CONTROL, batch: 08/26/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 62, updrs iii score off: 0, hoehn & yahr (off): 0, moca score: 28\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P03_F02_FRATRO129.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631279/suppl/GSM2631279_Neuro_Krainc_P03_F02_FRATRO129.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631280\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P03_F04_FRATRO131\n",
            " - geo_accession : GSM2631280\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO131, disease label: CONTROL, batch: 08/26/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 62, updrs iii score off: 0, hoehn & yahr (off): 0, moca score: 26\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P03_F04_FRATRO131.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631280/suppl/GSM2631280_Neuro_Krainc_P03_F04_FRATRO131.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631281\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P03_F06_FRATRO133\n",
            " - geo_accession : GSM2631281\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO133, disease label: CONTROL, batch: 08/26/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 78, updrs iii score off: 4, hoehn & yahr (off): 0, moca score: 21\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P03_F06_FRATRO133.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631281/suppl/GSM2631281_Neuro_Krainc_P03_F06_FRATRO133.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631282\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P03_F09_FRATRO136\n",
            " - geo_accession : GSM2631282\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO136, disease label: GPD, batch: 08/27/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): PARKIN, age at exam: 42, age at 1st symptoms: 37, updrs i: 2, updrs ii: 5, updrs iii score on: 11, updrs iv: 0, hoehn & yahr (on): 2, moca score: 27\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P03_F09_FRATRO136.CEL, Whole blood gene expression from GPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631282/suppl/GSM2631282_Neuro_Krainc_P03_F09_FRATRO136.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631283\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P04_B06_FRATRO186\n",
            " - geo_accession : GSM2631283\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO186, disease label: CONTROL, batch: 09/03/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 61, updrs iii score off: 0, hoehn & yahr (off): 0, moca score: 30\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P04_B06_FRATRO186.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631283/suppl/GSM2631283_Neuro_Krainc_P04_B06_FRATRO186.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631284\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P04_B12_FRATRO192\n",
            " - geo_accession : GSM2631284\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO192, disease label: CONTROL, batch: 09/03/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 61, updrs iii score off: 1, hoehn & yahr (off): 0, moca score: 28\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P04_B12_FRATRO192.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631284/suppl/GSM2631284_Neuro_Krainc_P04_B12_FRATRO192.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631285\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P04_E04_FRATRO220\n",
            " - geo_accession : GSM2631285\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO220, disease label: HD, batch: 09/02/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 49, age at 1st symptoms: 44, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P04_E04_FRATRO220.CEL, Whole blood gene expression from HD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631285/suppl/GSM2631285_Neuro_Krainc_P04_E04_FRATRO220.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631286\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P04_F05_FRATRO234_2\n",
            " - geo_accession : GSM2631286\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO234, disease label: HD, batch: 09/03/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 36, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P04_F05_FRATRO234_2.CEL, Whole blood gene expression from HD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631286/suppl/GSM2631286_Neuro_Krainc_P04_F05_FRATRO234_2.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631287\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P04_G10_FRATRO251\n",
            " - geo_accession : GSM2631287\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO251, disease label: CONTROL, batch: 09/03/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 63, updrs iii score off: 0, hoehn & yahr (off): 0, moca score: 19\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P04_G10_FRATRO251.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631287/suppl/GSM2631287_Neuro_Krainc_P04_G10_FRATRO251.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631288\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P04_H10_FRATRO265\n",
            " - geo_accession : GSM2631288\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO265, disease label: HD, batch: 09/03/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 30, age at 1st symptoms: ND, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P04_H10_FRATRO265.CEL, Whole blood gene expression from HD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631288/suppl/GSM2631288_Neuro_Krainc_P04_H10_FRATRO265.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631289\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P05_A03_FRATRO271\n",
            " - geo_accession : GSM2631289\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO271, disease label: CONTROL, batch: 09/02/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 62, updrs iii score off: 0, hoehn & yahr (off): 0, moca score: 26\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P05_A03_FRATRO271.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631289/suppl/GSM2631289_Neuro_Krainc_P05_A03_FRATRO271.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631290\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P05_A06_FRATRO274\n",
            " - geo_accession : GSM2631290\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO274, disease label: IPD, batch: 09/02/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 47, age at 1st symptoms: 45, updrs i: 0, updrs ii: 7, updrs iii score on: 15, updrs iv: 1, hoehn & yahr (on): 2, moca score: 29\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P05_A06_FRATRO274.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631290/suppl/GSM2631290_Neuro_Krainc_P05_A06_FRATRO274.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631291\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P05_A09_FRATRO278\n",
            " - geo_accession : GSM2631291\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO278, disease label: IPD, batch: 09/02/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 63, age at 1st symptoms: 60, updrs i: 1, updrs ii: 5, updrs iii score on: 11, updrs iv: 0, hoehn & yahr (off): 1, moca score: 25\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P05_A09_FRATRO278.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631291/suppl/GSM2631291_Neuro_Krainc_P05_A09_FRATRO278.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631292\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P05_B01_FRATRO283\n",
            " - geo_accession : GSM2631292\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO283, disease label: GPD, batch: 09/03/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): LRRK2, age at exam: 63, age at 1st symptoms: 53, updrs i: 3, updrs ii: 6, updrs iii score on: 16, updrs iv: 7, hoehn & yahr (on): 2.5, moca score: 27\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P05_B01_FRATRO283.CEL, Whole blood gene expression from GPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631292/suppl/GSM2631292_Neuro_Krainc_P05_B01_FRATRO283.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631293\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P05_B03_FRATRO285\n",
            " - geo_accession : GSM2631293\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO285, disease label: IPD, batch: 09/02/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 66, age at 1st symptoms: 63, updrs i: 0, updrs ii: 3, updrs iii score on: 2, updrs iv: 1, hoehn & yahr (on): ND, hoehn & yahr (off): ND, moca score: 29\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P05_B03_FRATRO285.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631293/suppl/GSM2631293_Neuro_Krainc_P05_B03_FRATRO285.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631294\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P05_B09_FRATRO292\n",
            " - geo_accession : GSM2631294\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO292, disease label: GPD, batch: 09/02/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): PARKIN, age at exam: 48, age at 1st symptoms: 38, updrs i: 2, updrs ii: 7, updrs iii score on: 9, updrs iv: 6, hoehn & yahr (on): 2, moca score: 30\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P05_B09_FRATRO292.CEL, Whole blood gene expression from GPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631294/suppl/GSM2631294_Neuro_Krainc_P05_B09_FRATRO292.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631295\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P05_B10_FRATRO293\n",
            " - geo_accession : GSM2631295\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO293, disease label: CONTROL, batch: 09/02/09_TRO, learning set: TEST, Sex: Female, genetic (mutated pd genes): NONE, age at exam: 60, updrs iii score off: 0, hoehn & yahr (off): 0, moca score: 30\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P05_B10_FRATRO293.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631295/suppl/GSM2631295_Neuro_Krainc_P05_B10_FRATRO293.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631296\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P05_C03_FRATRO298\n",
            " - geo_accession : GSM2631296\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO298, disease label: IPD, batch: 09/02/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 67, age at 1st symptoms: 57, updrs i: 4, updrs ii: 21, updrs iii score on: 26, updrs iv: 9, hoehn & yahr (on): 2.5, moca score: 20\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P05_C03_FRATRO298.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631296/suppl/GSM2631296_Neuro_Krainc_P05_C03_FRATRO298.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631297\n",
            "Metadata:\n",
            " - title : Neuro_Krainc_P05_C08_FRATRO303\n",
            " - geo_accession : GSM2631297\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO303, disease label: IPD, batch: 09/02/09_TRO, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at exam: 50, age at 1st symptoms: 46, updrs i: 0, updrs ii: 6, updrs iii score on: 2, updrs iv: 2, hoehn & yahr (on): 1, moca score: 30\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc_P05_C08_FRATRO303.CEL, Whole blood gene expression from IPD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631297/suppl/GSM2631297_Neuro_Krainc_P05_C08_FRATRO303.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631298\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_A07_L3663_01\n",
            " - geo_accession : GSM2631298\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3663, disease label: CONTROL, batch: 02/25/10_L, learning set: TEST, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_A07_L3663_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631298/suppl/GSM2631298_Neuro_Krainc-2_P02_A07_L3663_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631299\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_A12_L4021_01\n",
            " - geo_accession : GSM2631299\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L4021, disease label: MSA, batch: 02/25/10_L, learning set: TEST, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_A12_L4021_01.CEL, Whole blood gene expression from MSA patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631299/suppl/GSM2631299_Neuro_Krainc-2_P02_A12_L4021_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631300\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_B06_L1342_01\n",
            " - geo_accession : GSM2631300\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L1342, disease label: CONTROL, batch: 02/25/10_L, learning set: TEST, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_B06_L1342_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631300/suppl/GSM2631300_Neuro_Krainc-2_P02_B06_L1342_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631301\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_B09_L3686_01\n",
            " - geo_accession : GSM2631301\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3686, disease label: CONTROL, batch: 02/25/10_L, learning set: TEST, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_B09_L3686_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631301/suppl/GSM2631301_Neuro_Krainc-2_P02_B09_L3686_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631302\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_B11_L3669_01\n",
            " - geo_accession : GSM2631302\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3669, disease label: GENETIC_UNAFFECTED, batch: 02/25/10_L, learning set: TEST, genetic (mutated pd genes): LRRK2, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_B11_L3669_01.CEL, Whole blood gene expression from GENETIC_UNAFFECTED patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631302/suppl/GSM2631302_Neuro_Krainc-2_P02_B11_L3669_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631303\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_C05_L4106_01\n",
            " - geo_accession : GSM2631303\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L4106, disease label: DRD-DYT5, batch: 03/05/10_L, learning set: TEST, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_C05_L4106_01.CEL, Whole blood gene expression from DRD-DYT5 patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631303/suppl/GSM2631303_Neuro_Krainc-2_P02_C05_L4106_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631304\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_C06_L4025_01\n",
            " - geo_accession : GSM2631304\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L4025, disease label: CONTROL, batch: 03/05/10_L, learning set: TEST, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_C06_L4025_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631304/suppl/GSM2631304_Neuro_Krainc-2_P02_C06_L4025_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631305\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_C08_L3654_01\n",
            " - geo_accession : GSM2631305\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3654, disease label: CONTROL, batch: 03/05/10_L, learning set: TEST, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_C08_L3654_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631305/suppl/GSM2631305_Neuro_Krainc-2_P02_C08_L3654_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631306\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_C10_L3700_01\n",
            " - geo_accession : GSM2631306\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3700, disease label: CONTROL, batch: 03/05/10_L, learning set: TEST, genetic (mutated pd genes): NMF, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_C10_L3700_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631306/suppl/GSM2631306_Neuro_Krainc-2_P02_C10_L3700_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631307\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_D02_L3872_01\n",
            " - geo_accession : GSM2631307\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3872, disease label: GENETIC_UNAFFECTED, batch: 03/05/10_L, learning set: TEST, genetic (mutated pd genes): LRRK2, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_D02_L3872_01.CEL, Whole blood gene expression from GENETIC_UNAFFECTED patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631307/suppl/GSM2631307_Neuro_Krainc-2_P02_D02_L3872_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631308\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_D05_L4105_01\n",
            " - geo_accession : GSM2631308\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L4105, disease label: DRD-DYT5, batch: 03/05/10_L, learning set: TEST, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_D05_L4105_01.CEL, Whole blood gene expression from DRD-DYT5 patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631308/suppl/GSM2631308_Neuro_Krainc-2_P02_D05_L4105_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631309\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_D09_L2872_01\n",
            " - geo_accession : GSM2631309\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L2872, disease label: ATYPICAL_PD, batch: 03/05/10_L, learning set: TEST, Sex: Male, genetic (mutated pd genes): NONE, age at 1st symptoms: 64, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 9, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_D09_L2872_01.CEL, Whole blood gene expression from ATYPICAL_PD patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631309/suppl/GSM2631309_Neuro_Krainc-2_P02_D09_L2872_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631310\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_F02_L3668_01\n",
            " - geo_accession : GSM2631310\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L3668, disease label: CONTROL, batch: 03/05/10_L, learning set: TEST, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_F02_L3668_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631310/suppl/GSM2631310_Neuro_Krainc-2_P02_F02_L3668_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631311\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_F03_L4104_01\n",
            " - geo_accession : GSM2631311\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L4104, disease label: DRD-DYT5, batch: 03/05/10_L, learning set: TEST, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_F03_L4104_01.CEL, Whole blood gene expression from DRD-DYT5 patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631311/suppl/GSM2631311_Neuro_Krainc-2_P02_F03_L4104_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631312\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P02_F04_L4027_01\n",
            " - geo_accession : GSM2631312\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: L4027, disease label: CONTROL, batch: 03/05/10_L, learning set: TEST, updrs i: 0, updrs ii: 0, updrs iii score on: 0, updrs iii score off: 0, updrs iv: 0, hoehn & yahr (on): 0, hoehn & yahr (off): 0, moca score: 0\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P02_F04_L4027_01.CEL, Whole blood gene expression from CONTROL patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631312/suppl/GSM2631312_Neuro_Krainc-2_P02_F04_L4027_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631313\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_B10_FRA_TRO_357_01\n",
            " - geo_accession : GSM2631313\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO357, disease label: PSP, batch: 03/10/10_TRO, learning set: TEST, Sex: Male, age at exam: 63, age at 1st symptoms: 52, updrs iii score off: 26, hoehn & yahr (off): 2.5\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_B10_FRA_TRO_357_01.CEL, Whole blood gene expression from PSP patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631313/suppl/GSM2631313_Neuro_Krainc-2_P03_B10_FRA_TRO_357_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631314\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_B11_FRA_TRO_358_01\n",
            " - geo_accession : GSM2631314\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO358, disease label: PSP, batch: 03/10/10_TRO, learning set: TEST, Sex: Male, age at exam: 76, age at 1st symptoms: 66, updrs ii: 36, updrs iii score on: 75, updrs iv: 0, hoehn & yahr (on): 5\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_B11_FRA_TRO_358_01.CEL, Whole blood gene expression from PSP patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631314/suppl/GSM2631314_Neuro_Krainc-2_P03_B11_FRA_TRO_358_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "Name:  GSM2631315\n",
            "Metadata:\n",
            " - title : Neuro_Krainc-2_P03_C01_FRA_TRO_360_01\n",
            " - geo_accession : GSM2631315\n",
            " - status : Public on May 20 2017\n",
            " - submission_date : May 17 2017\n",
            " - last_update_date : Jan 23 2018\n",
            " - type : RNA\n",
            " - channel_count : 1\n",
            " - source_name_ch1 : Whole blood\n",
            " - organism_ch1 : Homo sapiens\n",
            " - taxid_ch1 : 9606\n",
            " - characteristics_ch1 : tissue: whole blood, subject_id: TRO360, disease label: MSA, batch: 03/10/10_TRO, learning set: TEST, Sex: Male, age at exam: 66, age at 1st symptoms: 60, updrs i: 0, updrs ii: 28, updrs iii score off: 43, updrs iv: 2, hoehn & yahr (off): 4\n",
            " - molecule_ch1 : total RNA\n",
            " - extract_protocol_ch1 : We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.\n",
            " - label_ch1 : biotin\n",
            " - label_protocol_ch1 : Samples were processed according to manufacturer’s protocols.\n",
            " - hyb_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - scan_protocol : Samples were processed according to manufacturer’s protocols.\n",
            " - description : Neuro_Krainc-2_P03_C01_FRA_TRO_360_01.CEL, Whole blood gene expression from MSA patient.\n",
            " - data_processing : Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.\n",
            " - platform_id : GPL570\n",
            " - contact_name : David,,Amar\n",
            " - contact_email : ddam.am@gmail.com\n",
            " - contact_department : Computer Science\n",
            " - contact_institute : Tel Aviv University\n",
            " - contact_address : Haim Levanon\n",
            " - contact_city : Tel Aviv\n",
            " - contact_zip/postal_code : 69978\n",
            " - contact_country : Israel\n",
            " - supplementary_file : ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2631nnn/GSM2631315/suppl/GSM2631315_Neuro_Krainc-2_P03_C01_FRA_TRO_360_01.CEL.gz\n",
            " - series_id : GSE99039\n",
            " - data_row_count : 54675\n",
            "\n",
            "GPL example:\n",
            "Name:  GPL570\n",
            "Metadata:\n",
            " - title : [HG-U133_Plus_2] Affymetrix Human Genome U133 Plus 2.0 Array\n",
            " - geo_accession : GPL570\n",
            " - status : Public on Nov 07 2003\n",
            " - submission_date : Nov 07 2003\n",
            " - last_update_date : Dec 14 2020\n",
            " - technology : in situ oligonucleotide\n",
            " - distribution : commercial\n",
            " - organism : Homo sapiens\n",
            " - taxid : 9606\n",
            " - manufacturer : Affymetrix\n",
            " - manufacture_protocol : see manufacturer's web site, , Complete coverage of the Human Genome U133 Set plus 6,500 additional genes for analysis of over 47,000 transcripts, All probe sets represented on the GeneChip Human Genome U133 Set are identically replicated on the GeneChip Human Genome U133 Plus 2.0 Array. The sequences from which these probe sets were derived were selected from GenBank®, dbEST, and RefSeq. The sequence clusters were created from the UniGene database (Build 133, April 20, 2001) and then refined by analysis and comparison with a number of other publicly available databases, including the Washington University EST trace repository and the University of California, Santa Cruz Golden-Path human genome database (April 2001 release)., In addition, there are 9,921 new probe sets representing approximately 6,500 new genes. These gene sequences were selected from GenBank, dbEST, and RefSeq. Sequence clusters were created from the UniGene database (Build 159, January 25, 2003) and refined by analysis and comparison with a number of other publicly available databases, including the Washington University EST trace repository and the NCBI human genome assembly (Build 31)., \n",
            " - description : Affymetrix submissions are typically submitted to GEO using the GEOarchive method described at http://www.ncbi.nlm.nih.gov/projects/geo/info/geo_affy.html, , June 03, 2009: annotation table updated with netaffx build 28, June 06, 2012: annotation table updated with netaffx build 32, June 23, 2016: annotation table updated with netaffx build 35\n",
            " - web_link : http://www.affymetrix.com/support/technical/byproduct.affx?product=hg-u133-plus, http://www.affymetrix.com/analysis/index.affx\n",
            " - contact_name : ,,Affymetrix, Inc.\n",
            " - contact_email : geo@ncbi.nlm.nih.gov, support@affymetrix.com\n",
            " - contact_phone : 888-362-2447\n",
            " - contact_institute : Affymetrix, Inc.\n",
            " - contact_address : \n",
            " - contact_city : Santa Clara\n",
            " - contact_state : CA\n",
            " - contact_zip/postal_code : 95051\n",
            " - contact_country : USA\n",
            " - contact_web_link : http://www.affymetrix.com/index.affx\n",
            " - relation : Alternative to: GPL4454 (Alternative CDF), Alternative to: GPL4866 (Alternative CDF), Alternative to: GPL5760 (Alternative CDF), Alternative to: GPL6671 (Alternative CDF), Alternative to: GPL6791 (Alternative CDF), Alternative to: GPL6879 (Alternative CDF), Alternative to: GPL7567 (Alternative CDF), Alternative to: GPL8019 (Alternative CDF), Alternative to: GPL8542 (Alternative CDF), Alternative to: GPL8715 (Alternative CDF), Alternative to: GPL8712 (Alternative CDF), Alternative to: GPL9102 (Probe Level Version), Alternative to: GPL9099 (Alternative CDF), Alternative to: GPL9101 (Alternative CDF), Alternative to: GPL9324 (Alternative CDF), Alternative to: GPL9486 (Alternative CDF), Alternative to: GPL9987 (Alternative CDF), Alternative to: GPL10175 (Alternative CDF), Alternative to: GPL10335 (Alternative CDF), Alternative to: GPL10371 (Alternative CDF), Alternative to: GPL10526 (Alternative CDF), Alternative to: GPL10881 (Alternative CDF), Alternative to: GPL10925 (Alternative CDF), Alternative to: GPL11084 (Alternative CDF), Alternative to: GPL11433 (Alternative CDF), Alternative to: GPL11670 (Alternative CDF), Alternative to: GPL13232 (Alternative CDF [HGU133Plus2_Hs_REFSEQ, Brainarray version 13.0.0]), Alternative to: GPL13303 (Alternative CDF.), Alternative to: GPL13916 ([Brainarray ENSG Version 14.1]), Alternative to: GPL14837 (Alternative CDF), Alternative to: GPL14877 ([Alternative CDF]), Alternative to: GPL15394 (Alternative CDF [HGU133Plus2_Hs_REFSEQ, Brainarray version 14.0]), Alternative to: GPL6732 (Alternative CDF [HGU133Plus2_Hs_REFSEQ, Brainarray version 7]), Alternative to: GPL15445 (Alternative CDF), Alternative to: GPL16066 (Alternative CDF), Alternative to: GPL16100 (Alternative CDF [GATExplorer_Ensembl v57]), Alternative to: GPL16268 (Alternative CDF [Brainarray ENSG Version 16]), Alternative to: GPL16273 (Alternative CDF [Brainarray ENSG Version 15]), Alternative to: GPL16311 (Alternative CDF [HGU133Plus2_Hs_ENTREZG Brainarray version 14.0.0]), Alternative to: GPL16356 (Alternative CDF [Brainarray HGU133Plus2_Hs_ENTREZG 15.0.0]), Alternative to: GPL16372 (Alternative CDF [Brainarray HGU133Plus2_Hs_UG v.13]), Alternative to: GPL17392 (Alternative CDF [HGU133Plus2_Hs_ENSG, Brainarray 17.1]), Alternative to: GPL17394 (Alternative CDF [HGU133Plus2_Hs_ENSG, Brainarray 13.0]), Alternative to: GPL17810 (Alternative CDF [HGU133Plus2_Hs_ENTREZG, Brainarray 16]), Alternative to: GPL17811 (Alternative CDF [HGU133Plus2_Hs_ENTREZG, Brainarray 17]), Alternative to: GPL17996 (Alternative CDF:HGU133Plus2_Hs_ENTREZG.cdf version 15.1.0), Alternative to: GPL18121 (custom CDF), Alternative to: GPL18756 (Alternative CDF [CDF: HGU133Plus2_Hs_ENTREZG_14.1.0]), Alternative to: GPL18850 (Alternative CDF), Alternative to: GPL18478 (Alternative CDF [Hs_ENTREZG_V13]), Alternative to: GPL19109 (Alternative CDF [Brainarray HGU133Plus2_Hs_ENTREZG_v18]), Alternative to: GPL19883 (Alternative CDF [Broad GenePattern]), Alternative to: GPL19918 (Gene symbol version, 10K), Alternative to: GPL20182 (Gene Symbol Version), Alternative to: GPL22321 (Alternative CDF [HGU133Plus2_Hs_ENTREZG, Brainarray 20]), Alternative to: GPL22945 (alternative), Alternative to: GPL23270 ((Alternative CDF [HGU133Plus2_Hs_ENTREZG, Brainarray 21]), Alternative to: GPL23432 (alternative CDF [HGU133Plus2_Hs_ENSG, Brainarray version 18.0.0]), Alternative to: GPL25740 (alternative), Alternative to: GPL28718 (Alternative CDF [HGU133Plus2_Hs_ENTREZG, Brainarray 23]), Alternative to: GPL29499 (Alternative CDF)\n",
            " - data_row_count : 54675\n",
            "558\n",
            "{'name': 'GSM2630758', 'metadata': {'title': ['E7R_039a01'], 'geo_accession': ['GSM2630758'], 'status': ['Public on May 20 2017'], 'submission_date': ['May 17 2017'], 'last_update_date': ['Jan 23 2018'], 'type': ['RNA'], 'channel_count': ['1'], 'source_name_ch1': ['Whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: whole blood', 'subject_id: L3012', 'disease label: CONTROL', 'batch: 01/15/08_L', 'learning set: TRAINING', 'Sex: Female', 'genetic (mutated pd genes): NONE', 'updrs i: 0', 'updrs ii: 0', 'updrs iii score on: 0', 'updrs iii score off: 0', 'updrs iv: 0', 'hoehn & yahr (on): 8', 'hoehn & yahr (off): 0', 'moca score: 29'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['We collected samples of venous blood using a standardized blood withdrawal protocol. PaxGene (Qiagen) and EDTA tubes were obtained. As participants were recruited at different time points, samples were frozen at -80° C first and RNA was extracted after patient recruitment at all centers was completed.'], 'label_ch1': ['biotin'], 'label_protocol_ch1': ['Samples were processed according to manufacturer’s protocols.'], 'hyb_protocol': ['Samples were processed according to manufacturer’s protocols.'], 'scan_protocol': ['Samples were processed according to manufacturer’s protocols.'], 'description': ['E7R_039a01_(HG-U133_Plus_2).CEL', 'Whole blood gene expression from CONTROL patient.'], 'data_processing': ['Data were preprocessed using RMA. Then the data were log2 transformed and quantile normalized. RMA was selected as it produced better patient correlations than MAS5 and GC-RMA.'], 'platform_id': ['GPL570'], 'contact_name': ['David,,Amar'], 'contact_email': ['ddam.am@gmail.com'], 'contact_department': ['Computer Science'], 'contact_institute': ['Tel Aviv University'], 'contact_address': ['Haim Levanon'], 'contact_city': ['Tel Aviv'], 'contact_zip/postal_code': ['69978'], 'contact_country': ['Israel'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2630nnn/GSM2630758/suppl/GSM2630758_E7R_039a01_HG-U133_Plus_2_.CEL.gz'], 'series_id': ['GSE99039'], 'data_row_count': ['54675']}}\n",
            "1\n",
            "{'name': 'GSM2631315', 'metadata': {'title': ['[HG-U133_Plus_2] Affymetrix Human Genome U133 Plus 2.0 Array'], 'geo_accession': ['GPL570'], 'status': ['Public on Nov 07 2003'], 'submission_date': ['Nov 07 2003'], 'last_update_date': ['Dec 14 2020'], 'technology': ['in situ oligonucleotide'], 'distribution': ['commercial'], 'organism': ['Homo sapiens'], 'taxid': ['9606'], 'manufacturer': ['Affymetrix'], 'manufacture_protocol': [\"see manufacturer's web site\", '', 'Complete coverage of the Human Genome U133 Set plus 6,500 additional genes for analysis of over 47,000 transcripts', 'All probe sets represented on the GeneChip Human Genome U133 Set are identically replicated on the GeneChip Human Genome U133 Plus 2.0 Array. The sequences from which these probe sets were derived were selected from GenBank®, dbEST, and RefSeq. The sequence clusters were created from the UniGene database (Build 133, April 20, 2001) and then refined by analysis and comparison with a number of other publicly available databases, including the Washington University EST trace repository and the University of California, Santa Cruz Golden-Path human genome database (April 2001 release).', 'In addition, there are 9,921 new probe sets representing approximately 6,500 new genes. These gene sequences were selected from GenBank, dbEST, and RefSeq. Sequence clusters were created from the UniGene database (Build 159, January 25, 2003) and refined by analysis and comparison with a number of other publicly available databases, including the Washington University EST trace repository and the NCBI human genome assembly (Build 31).', ''], 'description': ['Affymetrix submissions are typically submitted to GEO using the GEOarchive method described at http://www.ncbi.nlm.nih.gov/projects/geo/info/geo_affy.html', '', 'June 03, 2009: annotation table updated with netaffx build 28', 'June 06, 2012: annotation table updated with netaffx build 32', 'June 23, 2016: annotation table updated with netaffx build 35'], 'web_link': ['http://www.affymetrix.com/support/technical/byproduct.affx?product=hg-u133-plus', 'http://www.affymetrix.com/analysis/index.affx'], 'contact_name': [',,Affymetrix, Inc.'], 'contact_email': ['geo@ncbi.nlm.nih.gov, support@affymetrix.com'], 'contact_phone': ['888-362-2447'], 'contact_institute': ['Affymetrix, Inc.'], 'contact_address': [''], 'contact_city': ['Santa Clara'], 'contact_state': ['CA'], 'contact_zip/postal_code': ['95051'], 'contact_country': ['USA'], 'contact_web_link': ['http://www.affymetrix.com/index.affx'], 'relation': ['Alternative to: GPL4454 (Alternative CDF)', 'Alternative to: GPL4866 (Alternative CDF)', 'Alternative to: GPL5760 (Alternative CDF)', 'Alternative to: GPL6671 (Alternative CDF)', 'Alternative to: GPL6791 (Alternative CDF)', 'Alternative to: GPL6879 (Alternative CDF)', 'Alternative to: GPL7567 (Alternative CDF)', 'Alternative to: GPL8019 (Alternative CDF)', 'Alternative to: GPL8542 (Alternative CDF)', 'Alternative to: GPL8715 (Alternative CDF)', 'Alternative to: GPL8712 (Alternative CDF)', 'Alternative to: GPL9102 (Probe Level Version)', 'Alternative to: GPL9099 (Alternative CDF)', 'Alternative to: GPL9101 (Alternative CDF)', 'Alternative to: GPL9324 (Alternative CDF)', 'Alternative to: GPL9486 (Alternative CDF)', 'Alternative to: GPL9987 (Alternative CDF)', 'Alternative to: GPL10175 (Alternative CDF)', 'Alternative to: GPL10335 (Alternative CDF)', 'Alternative to: GPL10371 (Alternative CDF)', 'Alternative to: GPL10526 (Alternative CDF)', 'Alternative to: GPL10881 (Alternative CDF)', 'Alternative to: GPL10925 (Alternative CDF)', 'Alternative to: GPL11084 (Alternative CDF)', 'Alternative to: GPL11433 (Alternative CDF)', 'Alternative to: GPL11670 (Alternative CDF)', 'Alternative to: GPL13232 (Alternative CDF [HGU133Plus2_Hs_REFSEQ, Brainarray version 13.0.0])', 'Alternative to: GPL13303 (Alternative CDF.)', 'Alternative to: GPL13916 ([Brainarray ENSG Version 14.1])', 'Alternative to: GPL14837 (Alternative CDF)', 'Alternative to: GPL14877 ([Alternative CDF])', 'Alternative to: GPL15394 (Alternative CDF [HGU133Plus2_Hs_REFSEQ, Brainarray version 14.0])', 'Alternative to: GPL6732 (Alternative CDF [HGU133Plus2_Hs_REFSEQ, Brainarray version 7])', 'Alternative to: GPL15445 (Alternative CDF)', 'Alternative to: GPL16066 (Alternative CDF)', 'Alternative to: GPL16100 (Alternative CDF [GATExplorer_Ensembl v57])', 'Alternative to: GPL16268 (Alternative CDF [Brainarray ENSG Version 16])', 'Alternative to: GPL16273 (Alternative CDF [Brainarray ENSG Version 15])', 'Alternative to: GPL16311 (Alternative CDF [HGU133Plus2_Hs_ENTREZG Brainarray version 14.0.0])', 'Alternative to: GPL16356 (Alternative CDF [Brainarray HGU133Plus2_Hs_ENTREZG 15.0.0])', 'Alternative to: GPL16372 (Alternative CDF [Brainarray HGU133Plus2_Hs_UG v.13])', 'Alternative to: GPL17392 (Alternative CDF [HGU133Plus2_Hs_ENSG, Brainarray 17.1])', 'Alternative to: GPL17394 (Alternative CDF [HGU133Plus2_Hs_ENSG, Brainarray 13.0])', 'Alternative to: GPL17810 (Alternative CDF [HGU133Plus2_Hs_ENTREZG, Brainarray 16])', 'Alternative to: GPL17811 (Alternative CDF [HGU133Plus2_Hs_ENTREZG, Brainarray 17])', 'Alternative to: GPL17996 (Alternative CDF:HGU133Plus2_Hs_ENTREZG.cdf version 15.1.0)', 'Alternative to: GPL18121 (custom CDF)', 'Alternative to: GPL18756 (Alternative CDF [CDF: HGU133Plus2_Hs_ENTREZG_14.1.0])', 'Alternative to: GPL18850 (Alternative CDF)', 'Alternative to: GPL18478 (Alternative CDF [Hs_ENTREZG_V13])', 'Alternative to: GPL19109 (Alternative CDF [Brainarray HGU133Plus2_Hs_ENTREZG_v18])', 'Alternative to: GPL19883 (Alternative CDF [Broad GenePattern])', 'Alternative to: GPL19918 (Gene symbol version, 10K)', 'Alternative to: GPL20182 (Gene Symbol Version)', 'Alternative to: GPL22321 (Alternative CDF [HGU133Plus2_Hs_ENTREZG, Brainarray 20])', 'Alternative to: GPL22945 (alternative)', 'Alternative to: GPL23270 ((Alternative CDF [HGU133Plus2_Hs_ENTREZG, Brainarray 21])', 'Alternative to: GPL23432 (alternative CDF [HGU133Plus2_Hs_ENSG, Brainarray version 18.0.0])', 'Alternative to: GPL25740 (alternative)', 'Alternative to: GPL28718 (Alternative CDF [HGU133Plus2_Hs_ENTREZG, Brainarray 23])', 'Alternative to: GPL29499 (Alternative CDF)'], 'data_row_count': ['54675']}}\n"
          ]
        }
      ]
    }
  ]
}